declare -x AZ_BATCH_ACCOUNT_NAME="exp1hildebatch"
declare -x AZ_BATCH_ACCOUNT_URL="https://exp1hildebatch.southeastasia.batch.azure.com/"
declare -x AZ_BATCH_CERTIFICATES_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00027/certs"
declare -x AZ_BATCH_JOB_ID="svm"
declare -x AZ_BATCH_JOB_PREP_DIR="/mnt/batch/tasks/workitems/svm/job-1/jobpreparation"
declare -x AZ_BATCH_JOB_PREP_WORKING_DIR="/mnt/batch/tasks/workitems/svm/job-1/jobpreparation/wd"
declare -x AZ_BATCH_NODE_ID="tvm-2112069986_1-20180411t200903z"
declare -x AZ_BATCH_NODE_IS_DEDICATED="true"
declare -x AZ_BATCH_NODE_ROOT_DIR="/mnt/batch/tasks"
declare -x AZ_BATCH_NODE_SHARED_DIR="/mnt/batch/tasks/shared"
declare -x AZ_BATCH_NODE_STARTUP_DIR="/mnt/batch/tasks/startup"
declare -x AZ_BATCH_POOL_ID="experiment"
declare -x AZ_BATCH_TASK_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00027"
declare -x AZ_BATCH_TASK_ID="task-00027"
declare -x AZ_BATCH_TASK_USER="_azbatch"
declare -x AZ_BATCH_TASK_USER_IDENTITY="PoolAdmin"
declare -x AZ_BATCH_TASK_WORKING_DIR="/mnt/batch/tasks/workitems/svm/job-1/task-00027/wd"
declare -x HOME="/mnt/batch/tasks/workitems/svm/job-1/task-00027/wd"
declare -x OLDPWD
declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/mnt/batch/tasks/shared:/mnt/batch/tasks/workitems/svm/job-1/task-00027/wd"
declare -x PWD="/mnt/batch/tasks/workitems/svm/job-1/task-00027/wd"
declare -x SHLVL="1"
declare -x USER="_azbatch"
[2018-04-11 20:52:21] Results will not be logged locally.
[2018-04-11 20:52:21] Retrieving tasks...
[2018-04-11 20:52:21] Tasks will be retrieved based on provided task ids.
[2018-04-11 20:52:21] The following tasks were retrieved: [9981].
[2018-04-11 20:52:29] Downloaded task 9981.
[2018-04-11 20:52:29] Prepared pipelines.
[2018-04-11 20:52:29] Started run 1 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:29] Started run 2 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:29] Started run 3 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:29] Started run 4 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:29] Started run 6 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:29] Started run 5 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:29] Started run 7 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:29] Started run 8 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:34] Error in run 1: Run already exists in server. Run id(s): {9008513}
[2018-04-11 20:52:34] Error in run 2: Run already exists in server. Run id(s): {9008515}
[2018-04-11 20:52:34] Started run 9 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:34] Error in run 3: Run already exists in server. Run id(s): {9008511}
[2018-04-11 20:52:34] Started run 10 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:34] Started run 11 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:34] Error in run 4: Run already exists in server. Run id(s): {9008514}
[2018-04-11 20:52:34] Error in run 6: Run already exists in server. Run id(s): {9008531}
[2018-04-11 20:52:34] Error in run 5: Run already exists in server. Run id(s): {9008528}
[2018-04-11 20:52:34] Started run 12 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:34] Started run 13 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:35] Error in run 7: Run already exists in server. Run id(s): {9008533}
[2018-04-11 20:52:35] Error in run 8: Run already exists in server. Run id(s): {9008537}
[2018-04-11 20:52:34] Started run 14 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:35] Started run 16 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:35] Started run 15 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:39] Error in run 9: Run already exists in server. Run id(s): {9008553}
[2018-04-11 20:52:39] Started run 17 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:39] Error in run 10: Run already exists in server. Run id(s): {9008558}
[2018-04-11 20:52:39] Started run 18 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:39] Error in run 11: Run already exists in server. Run id(s): {9008563}
[2018-04-11 20:52:39] Started run 19 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:39] Error in run 12: Run already exists in server. Run id(s): {9008556}
[2018-04-11 20:52:39] Error in run 13: Run already exists in server. Run id(s): {9008578}
[2018-04-11 20:52:39] Started run 20 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:39] Started run 21 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:39] Error in run 14: Run already exists in server. Run id(s): {9008580}
[2018-04-11 20:52:39] Error in run 16: Run already exists in server. Run id(s): {9008585}
[2018-04-11 20:52:39] Started run 22 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:39] Started run 23 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:40] Error in run 15: Run already exists in server. Run id(s): {9008581}
[2018-04-11 20:52:40] Started run 24 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:43] Error in run 17: Run already exists in server. Run id(s): {9008603}
[2018-04-11 20:52:43] Started run 25 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:43] Error in run 18: Run already exists in server. Run id(s): {9008608}
[2018-04-11 20:52:43] Error in run 19: Run already exists in server. Run id(s): {9008610}
[2018-04-11 20:52:43] Started run 26 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:43] Started run 27 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:44] Error in run 20: Run already exists in server. Run id(s): {9008607}
[2018-04-11 20:52:44] Started run 28 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:44] Error in run 21: Run already exists in server. Run id(s): {9008625}
[2018-04-11 20:52:44] Error in run 22: Run already exists in server. Run id(s): {9008627}
[2018-04-11 20:52:44] Started run 29 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:44] Error in run 23: Run already exists in server. Run id(s): {9008628}
[2018-04-11 20:52:44] Error in run 24: Run already exists in server. Run id(s): {9008632}
[2018-04-11 20:52:44] Started run 30 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:44] Started run 31 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:44] Started run 32 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:48] Error in run 25: Run already exists in server. Run id(s): {9008645}
[2018-04-11 20:52:48] Error in run 26: Run already exists in server. Run id(s): {9008649}
[2018-04-11 20:52:48] Started run 33 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:48] Error in run 27: Run already exists in server. Run id(s): {9008651}
[2018-04-11 20:52:48] Started run 34 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:48] Started run 35 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:48] Error in run 28: Run already exists in server. Run id(s): {9008652}
[2018-04-11 20:52:48] Started run 36 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:49] Error in run 29: Run already exists in server. Run id(s): {9008665}
[2018-04-11 20:52:49] Error in run 30: Run already exists in server. Run id(s): {9008673}
[2018-04-11 20:52:49] Started run 37 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:49] Error in run 31: Run already exists in server. Run id(s): {9008677}
[2018-04-11 20:52:49] Started run 39 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:49] Error in run 32: Run already exists in server. Run id(s): {9008676}
[2018-04-11 20:52:49] Started run 38 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:49] Started run 40 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:53] Error in run 34: Run already exists in server. Run id(s): {9008693}
[2018-04-11 20:52:53] Error in run 33: Run already exists in server. Run id(s): {9008690}
[2018-04-11 20:52:53] Started run 41 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:53] Error in run 35: Run already exists in server. Run id(s): {9008699}
[2018-04-11 20:52:53] Started run 42 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:53] Started run 43 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:53] Error in run 36: Run already exists in server. Run id(s): {9008696}
[2018-04-11 20:52:53] Started run 44 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:53] Error in run 37: Run already exists in server. Run id(s): {9008716}
[2018-04-11 20:52:53] Error in run 38: Run already exists in server. Run id(s): {9008712}
[2018-04-11 20:52:53] Error in run 39: Run already exists in server. Run id(s): {9008717}
[2018-04-11 20:52:53] Started run 45 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:54] Error in run 40: Run already exists in server. Run id(s): {9008718}
[2018-04-11 20:52:53] Started run 46 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:54] Started run 48 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:53] Started run 47 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:57] Error in run 41: Run already exists in server. Run id(s): {9008731}
[2018-04-11 20:52:57] Started run 49 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:57] Error in run 42: Run already exists in server. Run id(s): {9008733}
[2018-04-11 20:52:57] Started run 50 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:58] Error in run 43: Run already exists in server. Run id(s): {9008738}
[2018-04-11 20:52:58] Started run 51 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:58] Error in run 44: Run already exists in server. Run id(s): {9008740}
[2018-04-11 20:52:58] Started run 52 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:58] Error in run 45: Run already exists in server. Run id(s): {9008746}
[2018-04-11 20:52:58] Started run 53 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:58] Error in run 46: Run already exists in server. Run id(s): {9008748}
[2018-04-11 20:52:58] Error in run 48: Run already exists in server. Run id(s): {9008758}
[2018-04-11 20:52:58] Started run 54 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:59] Error in run 47: Run already exists in server. Run id(s): {9008755}
[2018-04-11 20:52:58] Started run 55 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:52:59] Started run 56 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:02] Error in run 49: Run already exists in server. Run id(s): {9008767}
[2018-04-11 20:53:02] Started run 57 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:02] Error in run 50: Run already exists in server. Run id(s): {9008776}
[2018-04-11 20:53:02] Error in run 51: Run already exists in server. Run id(s): {9008780}
[2018-04-11 20:53:02] Started run 58 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:02] Error in run 52: Run already exists in server. Run id(s): {9008779}
[2018-04-11 20:53:02] Started run 59 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:02] Started run 60 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:03] Error in run 53: Run already exists in server. Run id(s): {9008792}
[2018-04-11 20:53:03] Error in run 54: Run already exists in server. Run id(s): {9008798}
[2018-04-11 20:53:03] Started run 61 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:03] Error in run 55: Run already exists in server. Run id(s): {9008802}
[2018-04-11 20:53:03] Started run 62 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:03] Error in run 56: Run already exists in server. Run id(s): {9008805}
[2018-04-11 20:53:03] Started run 63 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:03] Started run 64 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:07] Error in run 57: Run already exists in server. Run id(s): {9008815}
[2018-04-11 20:53:07] Started run 65 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:07] Error in run 59: Run already exists in server. Run id(s): {9008821}
[2018-04-11 20:53:07] Error in run 58: Run already exists in server. Run id(s): {9008819}
[2018-04-11 20:53:07] Started run 66 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:07] Error in run 60: Run already exists in server. Run id(s): {9008824}
[2018-04-11 20:53:07] Started run 67 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:07] Started run 68 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:08] Error in run 62: Run already exists in server. Run id(s): {9008843}
[2018-04-11 20:53:08] Error in run 61: Run already exists in server. Run id(s): {9008838}
[2018-04-11 20:53:08] Started run 69 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:08] Error in run 63: Run already exists in server. Run id(s): {9008845}
[2018-04-11 20:53:08] Started run 70 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:08] Error in run 64: Run already exists in server. Run id(s): {9008847}
[2018-04-11 20:53:08] Started run 71 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:08] Started run 72 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:11] Error in run 65: Run already exists in server. Run id(s): {9008853}
[2018-04-11 20:53:11] Started run 73 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:12] Error in run 67: Run already exists in server. Run id(s): {9008862}
[2018-04-11 20:53:12] Error in run 66: Run already exists in server. Run id(s): {9008857}
[2018-04-11 20:53:12] Error in run 68: Run already exists in server. Run id(s): {9008866}
[2018-04-11 20:53:12] Started run 74 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:12] Started run 75 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:12] Started run 76 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:12] Error in run 70: Run already exists in server. Run id(s): {9008879}
[2018-04-11 20:53:12] Error in run 69: Run already exists in server. Run id(s): {9008877}
[2018-04-11 20:53:12] Error in run 71: Run already exists in server. Run id(s): {9008880}
[2018-04-11 20:53:12] Started run 77 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:13] Error in run 72: Run already exists in server. Run id(s): {9008884}
[2018-04-11 20:53:12] Started run 78 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:13] Started run 80 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:12] Started run 79 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:16] Error in run 73: Run already exists in server. Run id(s): {9008892}
[2018-04-11 20:53:16] Started run 81 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:16] Error in run 74: Run already exists in server. Run id(s): {9008899}
[2018-04-11 20:53:16] Error in run 75: Run already exists in server. Run id(s): {9008902}
[2018-04-11 20:53:16] Started run 82 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:17] Error in run 76: Run already exists in server. Run id(s): {9008903}
[2018-04-11 20:53:16] Started run 83 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:17] Started run 84 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:17] Error in run 78: Run already exists in server. Run id(s): {9008919}
[2018-04-11 20:53:17] Error in run 77: Run already exists in server. Run id(s): {9008914}
[2018-04-11 20:53:17] Started run 85 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:17] Started run 86 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:17] Error in run 80: Run already exists in server. Run id(s): {9008923}
[2018-04-11 20:53:17] Error in run 79: Run already exists in server. Run id(s): {9008924}
[2018-04-11 20:53:17] Started run 87 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:17] Started run 88 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:20] Error in run 81: Run already exists in server. Run id(s): {9008932}
[2018-04-11 20:53:20] Started run 89 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:21] Error in run 83: Run already exists in server. Run id(s): {9008950}
[2018-04-11 20:53:21] Error in run 82: Run already exists in server. Run id(s): {9008945}
[2018-04-11 20:53:21] Error in run 84: Run already exists in server. Run id(s): {9008952}
[2018-04-11 20:53:21] Started run 91 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:21] Started run 90 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:21] Started run 92 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:22] Error in run 86: Run already exists in server. Run id(s): {9008967}
[2018-04-11 20:53:22] Error in run 87: Run already exists in server. Run id(s): {9008971}
[2018-04-11 20:53:22] Error in run 85: Run already exists in server. Run id(s): {9008960}
[2018-04-11 20:53:22] Started run 93 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:22] Started run 94 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:22] Error in run 88: Run already exists in server. Run id(s): {9008969}
[2018-04-11 20:53:22] Started run 95 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:22] Started run 96 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:25] Error in run 89: Run already exists in server. Run id(s): {9008976}
[2018-04-11 20:53:25] Started run 97 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:26] Error in run 91: Run already exists in server. Run id(s): {9008993}
[2018-04-11 20:53:26] Error in run 90: Run already exists in server. Run id(s): {9008992}
[2018-04-11 20:53:26] Started run 98 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:26] Started run 99 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:26] Error in run 92: Run already exists in server. Run id(s): {9008995}
[2018-04-11 20:53:26] Started run 100 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:27] Error in run 93: Run already exists in server. Run id(s): {9009002}
[2018-04-11 20:53:27] Error in run 94: Run already exists in server. Run id(s): {9009006}
[2018-04-11 20:53:27] Started run 101 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:27] Started run 102 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:27] Error in run 95: Run already exists in server. Run id(s): {9009010}
[2018-04-11 20:53:27] Error in run 96: Run already exists in server. Run id(s): {9009012}
[2018-04-11 20:53:27] Started run 103 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:27] Started run 104 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:30] Error in run 97: Run already exists in server. Run id(s): {9009018}
[2018-04-11 20:53:30] Started run 105 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:30] Error in run 98: Run already exists in server. Run id(s): {9009022}
[2018-04-11 20:53:30] Started run 106 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:31] Error in run 99: Run already exists in server. Run id(s): {9009026}
[2018-04-11 20:53:31] Error in run 100: Run already exists in server. Run id(s): {9009025}
[2018-04-11 20:53:31] Started run 107 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:31] Started run 108 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:31] Error in run 102: Run already exists in server. Run id(s): {9009039}
[2018-04-11 20:53:31] Error in run 101: Run already exists in server. Run id(s): {9009033}
[2018-04-11 20:53:31] Error in run 103: Run already exists in server. Run id(s): {9009045}
[2018-04-11 20:53:31] Started run 109 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:32] Error in run 104: Run already exists in server. Run id(s): {9009048}
[2018-04-11 20:53:31] Started run 110 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:32] Started run 112 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:31] Started run 111 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:34] Error in run 105: Run already exists in server. Run id(s): {9009056}
[2018-04-11 20:53:34] Started run 113 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:35] Error in run 106: Run already exists in server. Run id(s): {9009062}
[2018-04-11 20:53:35] Started run 114 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:35] Error in run 107: Run already exists in server. Run id(s): {9009068}
[2018-04-11 20:53:35] Error in run 108: Run already exists in server. Run id(s): {9009069}
[2018-04-11 20:53:35] Started run 115 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:35] Started run 116 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:36] Error in run 109: Run already exists in server. Run id(s): {9009074}
[2018-04-11 20:53:36] Error in run 110: Run already exists in server. Run id(s): {9009080}
[2018-04-11 20:53:36] Started run 117 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:36] Error in run 112: Run already exists in server. Run id(s): {9009092}
[2018-04-11 20:53:36] Started run 118 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:36] Error in run 111: Run already exists in server. Run id(s): {9009088}
[2018-04-11 20:53:36] Started run 119 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:36] Started run 120 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:39] Error in run 113: Run already exists in server. Run id(s): {9009099}
[2018-04-11 20:53:39] Started run 121 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:40] Error in run 114: Run already exists in server. Run id(s): {9009101}
[2018-04-11 20:53:40] Started run 122 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:40] Error in run 116: Run already exists in server. Run id(s): {9009113}
[2018-04-11 20:53:40] Error in run 115: Run already exists in server. Run id(s): {9009110}
[2018-04-11 20:53:40] Started run 123 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:40] Started run 124 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:41] Error in run 117: Run already exists in server. Run id(s): {9009116}
[2018-04-11 20:53:41] Error in run 119: Run already exists in server. Run id(s): {9009138}
[2018-04-11 20:53:41] Error in run 118: Run already exists in server. Run id(s): {9009124}
[2018-04-11 20:53:41] Started run 125 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:41] Error in run 120: Run already exists in server. Run id(s): {9009137}
[2018-04-11 20:53:41] Started run 126 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:41] Started run 127 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:41] Started run 128 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:44] Error in run 121: Run already exists in server. Run id(s): {9009140}
[2018-04-11 20:53:44] Started run 129 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:44] Error in run 122: Run already exists in server. Run id(s): {9009147}
[2018-04-11 20:53:44] Started run 130 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:45] Error in run 123: Run already exists in server. Run id(s): {9009152}
[2018-04-11 20:53:45] Error in run 124: Run already exists in server. Run id(s): {9009157}
[2018-04-11 20:53:45] Started run 131 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:45] Started run 132 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:45] Error in run 125: Run already exists in server. Run id(s): {9009161}
[2018-04-11 20:53:46] Error in run 126: Run already exists in server. Run id(s): {9009165}
[2018-04-11 20:53:45] Started run 133 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:46] Error in run 127: Run already exists in server. Run id(s): {9009174}
[2018-04-11 20:53:46] Started run 134 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:46] Error in run 128: Run already exists in server. Run id(s): {9009178}
[2018-04-11 20:53:46] Started run 136 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:46] Started run 135 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:49] Error in run 129: Run already exists in server. Run id(s): {9009182}
[2018-04-11 20:53:49] Started run 137 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:49] Error in run 130: Run already exists in server. Run id(s): {9009188}
[2018-04-11 20:53:49] Started run 138 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:50] Error in run 132: Run already exists in server. Run id(s): {9009195}
[2018-04-11 20:53:50] Error in run 131: Run already exists in server. Run id(s): {9009194}
[2018-04-11 20:53:50] Started run 139 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:50] Started run 140 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:50] Error in run 134: Run already exists in server. Run id(s): {9009209}
[2018-04-11 20:53:50] Error in run 133: Run already exists in server. Run id(s): {9009202}
[2018-04-11 20:53:50] Error in run 136: Run already exists in server. Run id(s): {9009218}
[2018-04-11 20:53:50] Started run 141 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:50] Error in run 135: Run already exists in server. Run id(s): {9009216}
[2018-04-11 20:53:50] Started run 142 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:50] Started run 143 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:50] Started run 144 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:53] Error in run 138: Run already exists in server. Run id(s): {9009232}
[2018-04-11 20:53:53] Error in run 137: Run already exists in server. Run id(s): {9009221}
[2018-04-11 20:53:53] Started run 145 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:53] Started run 146 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:54] Error in run 139: Run already exists in server. Run id(s): {9009235}
[2018-04-11 20:53:54] Error in run 140: Run already exists in server. Run id(s): {9009241}
[2018-04-11 20:53:54] Started run 147 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:54] Started run 148 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:55] Error in run 141: Run already exists in server. Run id(s): {9009244}
[2018-04-11 20:53:55] Error in run 142: Run already exists in server. Run id(s): {9009255}
[2018-04-11 20:53:55] Started run 149 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:55] Error in run 143: Run already exists in server. Run id(s): {9009263}
[2018-04-11 20:53:55] Started run 150 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:55] Error in run 144: Run already exists in server. Run id(s): {9009260}
[2018-04-11 20:53:55] Started run 151 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:55] Started run 152 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:58] Error in run 145: Run already exists in server. Run id(s): {9009266}
[2018-04-11 20:53:58] Error in run 146: Run already exists in server. Run id(s): {9009276}
[2018-04-11 20:53:58] Started run 153 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:58] Started run 154 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:59] Error in run 147: Run already exists in server. Run id(s): {9009280}
[2018-04-11 20:53:59] Error in run 148: Run already exists in server. Run id(s): {9009283}
[2018-04-11 20:53:59] Started run 155 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:53:59] Started run 156 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:00] Error in run 150: Run already exists in server. Run id(s): {9009297}
[2018-04-11 20:54:00] Error in run 149: Run already exists in server. Run id(s): {9009290}
[2018-04-11 20:54:00] Error in run 151: Run already exists in server. Run id(s): {9009300}
[2018-04-11 20:54:00] Error in run 152: Run already exists in server. Run id(s): {9009302}
[2018-04-11 20:54:00] Started run 157 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:00] Started run 159 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:00] Started run 158 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:00] Started run 160 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:03] Error in run 153: Run already exists in server. Run id(s): {9009306}
[2018-04-11 20:54:03] Error in run 154: Run already exists in server. Run id(s): {9009314}
[2018-04-11 20:54:03] Started run 161 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:03] Started run 162 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:04] Error in run 155: Run already exists in server. Run id(s): {9009321}
[2018-04-11 20:54:04] Error in run 156: Run already exists in server. Run id(s): {9009323}
[2018-04-11 20:54:04] Started run 163 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:04] Started run 164 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:04] Error in run 157: Run already exists in server. Run id(s): {9009328}
[2018-04-11 20:54:04] Error in run 159: Run already exists in server. Run id(s): {9009344}
[2018-04-11 20:54:05] Error in run 158: Run already exists in server. Run id(s): {9009335}
[2018-04-11 20:54:04] Started run 165 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:05] Started run 167 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:05] Error in run 160: Run already exists in server. Run id(s): {9009346}
[2018-04-11 20:54:04] Started run 166 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:05] Started run 168 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:08] Error in run 162: Run already exists in server. Run id(s): {9009360}
[2018-04-11 20:54:08] Error in run 161: Run already exists in server. Run id(s): {9009352}
[2018-04-11 20:54:08] Started run 169 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:08] Started run 170 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:09] Error in run 163: Run already exists in server. Run id(s): {9009362}
[2018-04-11 20:54:09] Error in run 164: Run already exists in server. Run id(s): {9009366}
[2018-04-11 20:54:09] Started run 171 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:09] Started run 172 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:09] Error in run 165: Run already exists in server. Run id(s): {9009371}
[2018-04-11 20:54:09] Error in run 167: Run already exists in server. Run id(s): {9009384}
[2018-04-11 20:54:09] Error in run 166: Run already exists in server. Run id(s): {9009379}
[2018-04-11 20:54:09] Error in run 168: Run already exists in server. Run id(s): {9009385}
[2018-04-11 20:54:09] Started run 173 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:09] Started run 174 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:09] Started run 175 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:09] Started run 176 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:12] Error in run 169: Run already exists in server. Run id(s): {9009394}
[2018-04-11 20:54:12] Started run 177 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:12] Error in run 170: Run already exists in server. Run id(s): {9009400}
[2018-04-11 20:54:12] Started run 178 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:13] Error in run 172: Run already exists in server. Run id(s): {9009404}
[2018-04-11 20:54:13] Error in run 171: Run already exists in server. Run id(s): {9009406}
[2018-04-11 20:54:13] Started run 179 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:13] Started run 180 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:14] Error in run 173: Run already exists in server. Run id(s): {9009412}
[2018-04-11 20:54:14] Error in run 174: Run already exists in server. Run id(s): {9009417}
[2018-04-11 20:54:14] Started run 181 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:14] Error in run 175: Run already exists in server. Run id(s): {9009419}
[2018-04-11 20:54:14] Started run 182 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:14] Started run 183 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:14] Error in run 176: Run already exists in server. Run id(s): {9009422}
[2018-04-11 20:54:14] Started run 184 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:17] Error in run 177: Run already exists in server. Run id(s): {9009432}
[2018-04-11 20:54:17] Started run 185 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:17] Error in run 178: Run already exists in server. Run id(s): {9009437}
[2018-04-11 20:54:17] Started run 186 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:18] Error in run 179: Run already exists in server. Run id(s): {9009438}
[2018-04-11 20:54:18] Error in run 180: Run already exists in server. Run id(s): {9009443}
[2018-04-11 20:54:18] Started run 187 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:18] Started run 188 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:19] Error in run 181: Run already exists in server. Run id(s): {9009453}
[2018-04-11 20:54:19] Started run 189 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:19] Error in run 182: Run already exists in server. Run id(s): {9009458}
[2018-04-11 20:54:19] Error in run 183: Run already exists in server. Run id(s): {9009461}
[2018-04-11 20:54:19] Started run 190 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:19] Error in run 184: Run already exists in server. Run id(s): {9009464}
[2018-04-11 20:54:19] Started run 191 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:19] Started run 192 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:21] Error in run 185: Run already exists in server. Run id(s): {9009472}
[2018-04-11 20:54:21] Started run 193 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:22] Error in run 186: Run already exists in server. Run id(s): {9009475}
[2018-04-11 20:54:22] Started run 194 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:23] Error in run 187: Run already exists in server. Run id(s): {9009479}
[2018-04-11 20:54:23] Error in run 188: Run already exists in server. Run id(s): {9009481}
[2018-04-11 20:54:23] Started run 195 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:23] Started run 196 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:23] Error in run 189: Run already exists in server. Run id(s): {9009486}
[2018-04-11 20:54:23] Started run 197 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:23] Error in run 190: Run already exists in server. Run id(s): {9009492}
[2018-04-11 20:54:23] Error in run 191: Run already exists in server. Run id(s): {9009498}
[2018-04-11 20:54:24] Error in run 192: Run already exists in server. Run id(s): {9009505}
[2018-04-11 20:54:23] Started run 198 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:24] Started run 200 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:23] Started run 199 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:26] Error in run 193: Run already exists in server. Run id(s): {9009508}
[2018-04-11 20:54:26] Started run 201 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:27] Error in run 194: Run already exists in server. Run id(s): {9009516}
[2018-04-11 20:54:27] Started run 202 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:27] Error in run 196: Run already exists in server. Run id(s): {9009528}
[2018-04-11 20:54:27] Error in run 195: Run already exists in server. Run id(s): {9009523}
[2018-04-11 20:54:27] Started run 203 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:27] Started run 204 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:28] Error in run 197: Run already exists in server. Run id(s): {9009531}
[2018-04-11 20:54:28] Started run 205 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:28] Error in run 200: Run already exists in server. Run id(s): {9009550}
[2018-04-11 20:54:28] Error in run 198: Run already exists in server. Run id(s): {9009537}
[2018-04-11 20:54:28] Error in run 199: Run already exists in server. Run id(s): {9009543}
[2018-04-11 20:54:28] Started run 206 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:28] Started run 208 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:28] Started run 207 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:31] Error in run 201: Run already exists in server. Run id(s): {9009553}
[2018-04-11 20:54:31] Started run 209 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:31] Error in run 202: Run already exists in server. Run id(s): {9009559}
[2018-04-11 20:54:31] Started run 210 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:32] Error in run 203: Run already exists in server. Run id(s): {9009565}
[2018-04-11 20:54:32] Error in run 204: Run already exists in server. Run id(s): {9009569}
[2018-04-11 20:54:32] Started run 211 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:32] Started run 212 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:32] Error in run 205: Run already exists in server. Run id(s): {9009572}
[2018-04-11 20:54:32] Started run 213 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:33] Error in run 208: Run already exists in server. Run id(s): {9009588}
[2018-04-11 20:54:33] Error in run 206: Run already exists in server. Run id(s): {9009578}
[2018-04-11 20:54:33] Started run 214 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:33] Error in run 207: Run already exists in server. Run id(s): {9009586}
[2018-04-11 20:54:33] Started run 215 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:33] Started run 216 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:35] Error in run 209: Run already exists in server. Run id(s): {9009590}
[2018-04-11 20:54:35] Started run 217 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:36] Error in run 210: Run already exists in server. Run id(s): {9009599}
[2018-04-11 20:54:36] Started run 218 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:37] Error in run 212: Run already exists in server. Run id(s): {9009608}
[2018-04-11 20:54:37] Error in run 211: Run already exists in server. Run id(s): {9009606}
[2018-04-11 20:54:37] Error in run 213: Run already exists in server. Run id(s): {9009613}
[2018-04-11 20:54:37] Started run 219 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:37] Started run 220 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:37] Started run 221 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:38] Error in run 215: Run already exists in server. Run id(s): {9009624}
[2018-04-11 20:54:38] Error in run 214: Run already exists in server. Run id(s): {9009620}
[2018-04-11 20:54:38] Error in run 216: Run already exists in server. Run id(s): {9009627}
[2018-04-11 20:54:38] Started run 222 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:38] Started run 224 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:38] Started run 223 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:40] Error in run 217: Run already exists in server. Run id(s): {9009629}
[2018-04-11 20:54:40] Started run 225 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:41] Error in run 218: Run already exists in server. Run id(s): {9009636}
[2018-04-11 20:54:41] Started run 226 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:42] Error in run 219: Run already exists in server. Run id(s): {9009642}
[2018-04-11 20:54:42] Error in run 220: Run already exists in server. Run id(s): {9009652}
[2018-04-11 20:54:42] Started run 227 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:42] Started run 228 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:42] Error in run 221: Run already exists in server. Run id(s): {9009654}
[2018-04-11 20:54:42] Started run 229 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:42] Error in run 222: Run already exists in server. Run id(s): {9009659}
[2018-04-11 20:54:42] Error in run 224: Run already exists in server. Run id(s): {9009667}
[2018-04-11 20:54:42] Started run 230 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:42] Started run 231 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:43] Error in run 223: Run already exists in server. Run id(s): {9009664}
[2018-04-11 20:54:43] Started run 232 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:45] Error in run 225: Run already exists in server. Run id(s): {9009674}
[2018-04-11 20:54:45] Started run 233 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:45] Error in run 226: Run already exists in server. Run id(s): {9009681}
[2018-04-11 20:54:45] Started run 234 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:46] Error in run 228: Run already exists in server. Run id(s): {9009688}
[2018-04-11 20:54:46] Error in run 227: Run already exists in server. Run id(s): {9009684}
[2018-04-11 20:54:46] Started run 235 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:47] Error in run 229: Run already exists in server. Run id(s): {9009692}
[2018-04-11 20:54:46] Started run 236 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:47] Started run 237 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:47] Error in run 230: Run already exists in server. Run id(s): {9009698}
[2018-04-11 20:54:47] Started run 238 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:47] Error in run 231: Run already exists in server. Run id(s): {9009702}
[2018-04-11 20:54:47] Error in run 232: Run already exists in server. Run id(s): {9009708}
[2018-04-11 20:54:47] Started run 239 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:47] Started run 240 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:50] Error in run 233: Run already exists in server. Run id(s): {9009714}
[2018-04-11 20:54:50] Started run 241 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:50] Error in run 234: Run already exists in server. Run id(s): {9009719}
[2018-04-11 20:54:50] Started run 242 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:51] Error in run 235: Run already exists in server. Run id(s): {9009725}
[2018-04-11 20:54:51] Error in run 236: Run already exists in server. Run id(s): {9009727}
[2018-04-11 20:54:51] Error in run 237: Run already exists in server. Run id(s): {9009736}
[2018-04-11 20:54:51] Started run 243 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:51] Started run 244 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:51] Started run 245 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:52] Error in run 238: Run already exists in server. Run id(s): {9009741}
[2018-04-11 20:54:52] Started run 246 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:52] Error in run 239: Run already exists in server. Run id(s): {9009745}
[2018-04-11 20:54:52] Error in run 240: Run already exists in server. Run id(s): {9009749}
[2018-04-11 20:54:52] Started run 247 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:52] Started run 248 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:54] Error in run 241: Run already exists in server. Run id(s): {9009756}
[2018-04-11 20:54:54] Started run 249 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:55] Error in run 242: Run already exists in server. Run id(s): {9009761}
[2018-04-11 20:54:55] Started run 250 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:56] Error in run 243: Run already exists in server. Run id(s): {9009763}
[2018-04-11 20:54:56] Error in run 244: Run already exists in server. Run id(s): {9009767}
[2018-04-11 20:54:56] Error in run 245: Run already exists in server. Run id(s): {9009775}
[2018-04-11 20:54:56] Started run 251 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:56] Error in run 246: Run already exists in server. Run id(s): {9009780}
[2018-04-11 20:54:56] Started run 252 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:56] Started run 254 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:56] Started run 253 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:56] Error in run 247: Run already exists in server. Run id(s): {9009781}
[2018-04-11 20:54:56] Started run 255 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:57] Error in run 248: Run already exists in server. Run id(s): {9009784}
[2018-04-11 20:54:57] Started run 256 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:54:59] Error in run 249: Run already exists in server. Run id(s): {9009795}
[2018-04-11 20:54:59] Started run 257 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:00] Error in run 250: Run already exists in server. Run id(s): {9009799}
[2018-04-11 20:55:00] Started run 258 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:01] Error in run 251: Run already exists in server. Run id(s): {9009804}
[2018-04-11 20:55:01] Error in run 252: Run already exists in server. Run id(s): {9009806}
[2018-04-11 20:55:01] Started run 259 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:01] Error in run 254: Run already exists in server. Run id(s): {9009821}
[2018-04-11 20:55:01] Started run 260 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:01] Error in run 253: Run already exists in server. Run id(s): {9009812}
[2018-04-11 20:55:01] Started run 261 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:01] Started run 262 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:01] Error in run 255: Run already exists in server. Run id(s): {9009825}
[2018-04-11 20:55:01] Started run 263 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:01] Error in run 256: Run already exists in server. Run id(s): {9009827}
[2018-04-11 20:55:01] Started run 264 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:04] Error in run 257: Run already exists in server. Run id(s): {9009837}
[2018-04-11 20:55:04] Started run 265 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:05] Error in run 258: Run already exists in server. Run id(s): {9009839}
[2018-04-11 20:55:05] Started run 266 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:05] Error in run 259: Run already exists in server. Run id(s): {9009842}
[2018-04-11 20:55:05] Error in run 261: Run already exists in server. Run id(s): {9009851}
[2018-04-11 20:55:05] Error in run 260: Run already exists in server. Run id(s): {9009845}
[2018-04-11 20:55:05] Started run 267 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:06] Error in run 262: Run already exists in server. Run id(s): {9009857}
[2018-04-11 20:55:06] Started run 270 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:05] Started run 268 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:06] Error in run 263: Run already exists in server. Run id(s): {9009860}
[2018-04-11 20:55:05] Started run 269 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:06] Started run 271 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:06] Error in run 264: Run already exists in server. Run id(s): {9009869}
[2018-04-11 20:55:06] Started run 272 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:09] Error in run 265: Run already exists in server. Run id(s): {9009878}
[2018-04-11 20:55:09] Started run 273 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:09] Error in run 266: Run already exists in server. Run id(s): {9009877}
[2018-04-11 20:55:09] Started run 274 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:10] Error in run 267: Run already exists in server. Run id(s): {9009883}
[2018-04-11 20:55:10] Error in run 270: Run already exists in server. Run id(s): {9009904}
[2018-04-11 20:55:10] Started run 275 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:10] Started run 276 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:10] Error in run 268: Run already exists in server. Run id(s): {9009890}
[2018-04-11 20:55:10] Started run 277 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:11] Error in run 269: Run already exists in server. Run id(s): {9009901}
[2018-04-11 20:55:11] Started run 278 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:11] Error in run 271: Run already exists in server. Run id(s): {9009908}
[2018-04-11 20:55:11] Started run 279 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:11] Error in run 272: Run already exists in server. Run id(s): {9009911}
[2018-04-11 20:55:11] Started run 280 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:13] Error in run 273: Run already exists in server. Run id(s): {9009925}
[2018-04-11 20:55:13] Started run 281 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:14] Error in run 274: Run already exists in server. Run id(s): {9009927}
[2018-04-11 20:55:14] Started run 282 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:15] Error in run 275: Run already exists in server. Run id(s): {9009931, 9014519}
[2018-04-11 20:55:15] Error in run 277: Run already exists in server. Run id(s): {9009945}
[2018-04-11 20:55:15] Error in run 276: Run already exists in server. Run id(s): {9009935}
[2018-04-11 20:55:15] Started run 283 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:15] Started run 284 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:15] Started run 285 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:15] Error in run 278: Run already exists in server. Run id(s): {9009950}
[2018-04-11 20:55:15] Started run 286 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:16] Error in run 279: Run already exists in server. Run id(s): {9009952}
[2018-04-11 20:55:16] Error in run 280: Run already exists in server. Run id(s): {9009957}
[2018-04-11 20:55:16] Started run 287 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:16] Started run 288 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:18] Error in run 281: Run already exists in server. Run id(s): {9009966}
[2018-04-11 20:55:18] Started run 289 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:19] Error in run 282: Run already exists in server. Run id(s): {9009970}
[2018-04-11 20:55:19] Started run 290 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:19] Error in run 283: Run already exists in server. Run id(s): {9009972}
[2018-04-11 20:55:20] Error in run 284: Run already exists in server. Run id(s): {9009975}
[2018-04-11 20:55:19] Started run 291 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:20] Error in run 285: Run already exists in server. Run id(s): {9009987}
[2018-04-11 20:55:20] Started run 293 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:20] Error in run 286: Run already exists in server. Run id(s): {9009989}
[2018-04-11 20:55:20] Started run 292 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:20] Started run 294 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:20] Error in run 287: Run already exists in server. Run id(s): {9009995}
[2018-04-11 20:55:20] Error in run 288: Run already exists in server. Run id(s): {9009992}
[2018-04-11 20:55:20] Started run 295 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:20] Started run 296 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:23] Error in run 289: Run already exists in server. Run id(s): {9010008}
[2018-04-11 20:55:23] Started run 297 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:23] Error in run 290: Run already exists in server. Run id(s): {9010009}
[2018-04-11 20:55:23] Started run 298 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:24] Error in run 291: Run already exists in server. Run id(s): {9010011}
[2018-04-11 20:55:24] Started run 299 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:24] Error in run 293: Run already exists in server. Run id(s): {9010026}
[2018-04-11 20:55:24] Error in run 292: Run already exists in server. Run id(s): {9010014}
[2018-04-11 20:55:24] Started run 300 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:24] Started run 301 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:25] Error in run 294: Run already exists in server. Run id(s): {9010028}
[2018-04-11 20:55:25] Started run 302 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:25] Error in run 295: Run already exists in server. Run id(s): {9010030}
[2018-04-11 20:55:25] Started run 303 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:25] Error in run 296: Run already exists in server. Run id(s): {9010031}
[2018-04-11 20:55:25] Started run 304 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:28] Error in run 297: Run already exists in server. Run id(s): {9010042}
[2018-04-11 20:55:28] Started run 305 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:28] Error in run 298: Run already exists in server. Run id(s): {9010045}
[2018-04-11 20:55:28] Started run 306 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:29] Error in run 299: Run already exists in server. Run id(s): {9010043}
[2018-04-11 20:55:29] Started run 307 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:29] Error in run 300: Run already exists in server. Run id(s): {9010046}
[2018-04-11 20:55:29] Error in run 301: Run already exists in server. Run id(s): {9010057}
[2018-04-11 20:55:29] Started run 308 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:29] Error in run 302: Run already exists in server. Run id(s): {9010058}
[2018-04-11 20:55:29] Started run 309 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:29] Started run 310 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:30] Error in run 303: Run already exists in server. Run id(s): {9010060}
[2018-04-11 20:55:30] Started run 311 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:30] Error in run 304: Run already exists in server. Run id(s): {9010059}
[2018-04-11 20:55:30] Started run 312 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:32] Error in run 305: Run already exists in server. Run id(s): {9010069}
[2018-04-11 20:55:32] Started run 313 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:33] Error in run 306: Run already exists in server. Run id(s): {9010072}
[2018-04-11 20:55:33] Started run 314 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:33] Error in run 307: Run already exists in server. Run id(s): {9010073}
[2018-04-11 20:55:33] Started run 315 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:34] Error in run 308: Run already exists in server. Run id(s): {9010074, 9014666}
[2018-04-11 20:55:34] Started run 316 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:34] Error in run 309: Run already exists in server. Run id(s): {9010080}
[2018-04-11 20:55:34] Started run 317 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:34] Error in run 310: Run already exists in server. Run id(s): {9010088}
[2018-04-11 20:55:34] Started run 318 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:34] Error in run 311: Run already exists in server. Run id(s): {9010094}
[2018-04-11 20:55:34] Started run 319 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:35] Error in run 312: Run already exists in server. Run id(s): {9010095}
[2018-04-11 20:55:35] Started run 320 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:37] Error in run 313: Run already exists in server. Run id(s): {9010107}
[2018-04-11 20:55:37] Started run 321 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:38] Error in run 314: Run already exists in server. Run id(s): {9010110}
[2018-04-11 20:55:38] Started run 322 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:38] Error in run 315: Run already exists in server. Run id(s): {9010113}
[2018-04-11 20:55:38] Started run 323 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:38] Error in run 316: Run already exists in server. Run id(s): {9010115}
[2018-04-11 20:55:38] Started run 324 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:38] Error in run 317: Run already exists in server. Run id(s): {9010124}
[2018-04-11 20:55:38] Started run 325 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:39] Error in run 318: Run already exists in server. Run id(s): {9010129}
[2018-04-11 20:55:39] Started run 326 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:39] Error in run 319: Run already exists in server. Run id(s): {9010133}
[2018-04-11 20:55:39] Started run 327 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:39] Error in run 320: Run already exists in server. Run id(s): {9010135}
[2018-04-11 20:55:39] Started run 328 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:42] Error in run 321: Run already exists in server. Run id(s): {9010146}
[2018-04-11 20:55:42] Started run 329 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:42] Error in run 322: Run already exists in server. Run id(s): {9010150}
[2018-04-11 20:55:42] Started run 330 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:43] Error in run 323: Run already exists in server. Run id(s): {9010152}
[2018-04-11 20:55:43] Started run 331 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:43] Error in run 324: Run already exists in server. Run id(s): {9010154}
[2018-04-11 20:55:43] Started run 332 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:43] Error in run 325: Run already exists in server. Run id(s): {9010166}
[2018-04-11 20:55:43] Error in run 326: Run already exists in server. Run id(s): {9010168}
[2018-04-11 20:55:43] Error in run 327: Run already exists in server. Run id(s): {9010175}
[2018-04-11 20:55:43] Started run 333 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:43] Started run 335 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:43] Started run 334 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:44] Error in run 328: Run already exists in server. Run id(s): {9010172}
[2018-04-11 20:55:44] Started run 336 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:46] Error in run 329: Run already exists in server. Run id(s): {9010182}
[2018-04-11 20:55:46] Started run 337 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:47] Error in run 330: Run already exists in server. Run id(s): {9010185}
[2018-04-11 20:55:47] Started run 338 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:47] Error in run 331: Run already exists in server. Run id(s): {9010188}
[2018-04-11 20:55:47] Started run 339 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:47] Error in run 332: Run already exists in server. Run id(s): {9010194}
[2018-04-11 20:55:47] Started run 340 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:48] Error in run 335: Run already exists in server. Run id(s): {9010211}
[2018-04-11 20:55:48] Error in run 333: Run already exists in server. Run id(s): {9010203}
[2018-04-11 20:55:48] Started run 341 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:48] Error in run 334: Run already exists in server. Run id(s): {9010206}
[2018-04-11 20:55:48] Started run 342 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:48] Started run 343 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:49] Error in run 336: Run already exists in server. Run id(s): {9010214}
[2018-04-11 20:55:49] Started run 344 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:51] Error in run 337: Run already exists in server. Run id(s): {9010223}
[2018-04-11 20:55:51] Started run 345 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:51] Error in run 338: Run already exists in server. Run id(s): {9010229}
[2018-04-11 20:55:51] Started run 346 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:52] Error in run 339: Run already exists in server. Run id(s): {9010233}
[2018-04-11 20:55:52] Error in run 340: Run already exists in server. Run id(s): {9010234}
[2018-04-11 20:55:52] Started run 347 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:52] Started run 348 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:53] Error in run 342: Run already exists in server. Run id(s): {9010250}
[2018-04-11 20:55:53] Error in run 341: Run already exists in server. Run id(s): {9010247}
[2018-04-11 20:55:53] Error in run 343: Run already exists in server. Run id(s): {9010255}
[2018-04-11 20:55:53] Started run 349 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:53] Started run 350 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:53] Started run 351 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:54] Error in run 344: Run already exists in server. Run id(s): {9010258}
[2018-04-11 20:55:54] Started run 352 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:56] Error in run 346: Run already exists in server. Run id(s): {9010270}
[2018-04-11 20:55:56] Error in run 345: Run already exists in server. Run id(s): {9010267}
[2018-04-11 20:55:56] Started run 353 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:56] Started run 354 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:57] Error in run 347: Run already exists in server. Run id(s): {9010277}
[2018-04-11 20:55:57] Error in run 348: Run already exists in server. Run id(s): {9010281}
[2018-04-11 20:55:57] Started run 355 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:57] Started run 356 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:57] Error in run 349: Run already exists in server. Run id(s): {9010289}
[2018-04-11 20:55:57] Error in run 350: Run already exists in server. Run id(s): {9010292}
[2018-04-11 20:55:57] Started run 357 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:58] Error in run 351: Run already exists in server. Run id(s): {9010298}
[2018-04-11 20:55:57] Started run 358 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:58] Started run 359 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:55:58] Error in run 352: Run already exists in server. Run id(s): {9010302}
[2018-04-11 20:55:58] Started run 360 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:01] Error in run 353: Run already exists in server. Run id(s): {9010310}
[2018-04-11 20:56:01] Error in run 354: Run already exists in server. Run id(s): {9010311}
[2018-04-11 20:56:01] Started run 361 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:01] Started run 362 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:01] Error in run 355: Run already exists in server. Run id(s): {9010320}
[2018-04-11 20:56:01] Error in run 356: Run already exists in server. Run id(s): {9010323}
[2018-04-11 20:56:01] Started run 363 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:01] Started run 364 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:02] Error in run 358: Run already exists in server. Run id(s): {9010337}
[2018-04-11 20:56:02] Error in run 357: Run already exists in server. Run id(s): {9010333}
[2018-04-11 20:56:02] Error in run 359: Run already exists in server. Run id(s): {9010340}
[2018-04-11 20:56:02] Started run 365 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:02] Started run 366 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:02] Started run 367 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:03] Error in run 360: Run already exists in server. Run id(s): {9010347}
[2018-04-11 20:56:03] Started run 368 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:05] Error in run 361: Run already exists in server. Run id(s): {9010358}
[2018-04-11 20:56:05] Error in run 362: Run already exists in server. Run id(s): {9010360}
[2018-04-11 20:56:05] Started run 369 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:05] Started run 370 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:06] Error in run 364: Run already exists in server. Run id(s): {9010365}
[2018-04-11 20:56:06] Error in run 363: Run already exists in server. Run id(s): {9010361}
[2018-04-11 20:56:06] Started run 371 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:06] Started run 372 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:07] Error in run 365: Run already exists in server. Run id(s): {9010376}
[2018-04-11 20:56:07] Started run 373 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:07] Error in run 366: Run already exists in server. Run id(s): {9010374}
[2018-04-11 20:56:07] Started run 374 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:07] Error in run 367: Run already exists in server. Run id(s): {9010379}
[2018-04-11 20:56:07] Started run 375 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:08] Error in run 368: Run already exists in server. Run id(s): {9010392}
[2018-04-11 20:56:08] Started run 376 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:10] Error in run 369: Run already exists in server. Run id(s): {9010401}
[2018-04-11 20:56:10] Error in run 370: Run already exists in server. Run id(s): {9010402}
[2018-04-11 20:56:10] Started run 377 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:10] Started run 378 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:11] Error in run 371: Run already exists in server. Run id(s): {9010406}
[2018-04-11 20:56:11] Error in run 372: Run already exists in server. Run id(s): {9010414}
[2018-04-11 20:56:11] Started run 379 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:11] Started run 380 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:12] Error in run 373: Run already exists in server. Run id(s): {9010420}
[2018-04-11 20:56:12] Error in run 375: Run already exists in server. Run id(s): {9010432}
[2018-04-11 20:56:12] Error in run 374: Run already exists in server. Run id(s): {9010424}
[2018-04-11 20:56:12] Started run 381 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:12] Started run 382 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:12] Started run 383 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:12] Error in run 376: Run already exists in server. Run id(s): {9010437}
[2018-04-11 20:56:12] Started run 384 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:15] Error in run 377: Run already exists in server. Run id(s): {9010447}
[2018-04-11 20:56:15] Started run 385 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:15] Error in run 378: Run already exists in server. Run id(s): {9010449}
[2018-04-11 20:56:15] Started run 386 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:16] Error in run 380: Run already exists in server. Run id(s): {9010454}
[2018-04-11 20:56:16] Error in run 379: Run already exists in server. Run id(s): {9010451}
[2018-04-11 20:56:16] Started run 387 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:16] Started run 388 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:16] Error in run 381: Run already exists in server. Run id(s): {9010467}
[2018-04-11 20:56:16] Error in run 382: Run already exists in server. Run id(s): {9010469}
[2018-04-11 20:56:16] Started run 389 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:17] Error in run 383: Run already exists in server. Run id(s): {9010470}
[2018-04-11 20:56:16] Started run 390 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:17] Started run 391 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:17] Error in run 384: Run already exists in server. Run id(s): {9010476}
[2018-04-11 20:56:17] Started run 392 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:19] Error in run 385: Run already exists in server. Run id(s): {9010489}
[2018-04-11 20:56:19] Started run 393 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:20] Error in run 386: Run already exists in server. Run id(s): {9010490}
[2018-04-11 20:56:20] Started run 394 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:20] Error in run 387: Run already exists in server. Run id(s): {9010493}
[2018-04-11 20:56:20] Error in run 388: Run already exists in server. Run id(s): {9010500}
[2018-04-11 20:56:20] Started run 395 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:20] Started run 396 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:21] Error in run 389: Run already exists in server. Run id(s): {9010511}
[2018-04-11 20:56:21] Error in run 390: Run already exists in server. Run id(s): {9010515}
[2018-04-11 20:56:21] Error in run 391: Run already exists in server. Run id(s): {9010517}
[2018-04-11 20:56:21] Started run 397 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:21] Started run 398 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:21] Started run 399 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:22] Error in run 392: Run already exists in server. Run id(s): {9010518}
[2018-04-11 20:56:22] Started run 400 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:24] Error in run 393: Run already exists in server. Run id(s): {9010529}
[2018-04-11 20:56:24] Started run 401 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:24] Error in run 394: Run already exists in server. Run id(s): {9010532}
[2018-04-11 20:56:24] Started run 402 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:25] Error in run 395: Run already exists in server. Run id(s): {9010534}
[2018-04-11 20:56:25] Error in run 396: Run already exists in server. Run id(s): {9010538}
[2018-04-11 20:56:25] Started run 403 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:25] Started run 404 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:26] Error in run 397: Run already exists in server. Run id(s): {9010549}
[2018-04-11 20:56:26] Error in run 398: Run already exists in server. Run id(s): {9010552}
[2018-04-11 20:56:26] Started run 405 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:26] Error in run 399: Run already exists in server. Run id(s): {9010555}
[2018-04-11 20:56:26] Started run 407 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:26] Started run 406 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:27] Error in run 400: Run already exists in server. Run id(s): {9010560}
[2018-04-11 20:56:27] Started run 408 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:29] Error in run 401: Run already exists in server. Run id(s): {9010572}
[2018-04-11 20:56:29] Started run 409 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:29] Error in run 402: Run already exists in server. Run id(s): {9010575}
[2018-04-11 20:56:29] Started run 410 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:30] Error in run 403: Run already exists in server. Run id(s): {9010578}
[2018-04-11 20:56:30] Error in run 404: Run already exists in server. Run id(s): {9010582}
[2018-04-11 20:56:30] Started run 411 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:30] Started run 412 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:31] Error in run 406: Run already exists in server. Run id(s): {9010598}
[2018-04-11 20:56:31] Error in run 405: Run already exists in server. Run id(s): {9010595}
[2018-04-11 20:56:31] Error in run 407: Run already exists in server. Run id(s): {9010602}
[2018-04-11 20:56:31] Started run 413 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:31] Started run 415 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:31] Started run 414 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:32] Error in run 408: Run already exists in server. Run id(s): {9010604}
[2018-04-11 20:56:32] Started run 416 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:33] Error in run 409: Run already exists in server. Run id(s): {9010620}
[2018-04-11 20:56:33] Started run 417 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:34] Error in run 410: Run already exists in server. Run id(s): {9010623}
[2018-04-11 20:56:34] Started run 418 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:35] Error in run 411: Run already exists in server. Run id(s): {9010629}
[2018-04-11 20:56:35] Error in run 412: Run already exists in server. Run id(s): {9010628}
[2018-04-11 20:56:35] Started run 419 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:35] Started run 420 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:35] Error in run 413: Run already exists in server. Run id(s): {9010648}
[2018-04-11 20:56:35] Error in run 415: Run already exists in server. Run id(s): {9010656}
[2018-04-11 20:56:35] Started run 421 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:36] Error in run 414: Run already exists in server. Run id(s): {9010652}
[2018-04-11 20:56:35] Started run 422 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:36] Started run 423 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:36] Error in run 416: Run already exists in server. Run id(s): {9010658}
[2018-04-11 20:56:36] Started run 424 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:38] Error in run 417: Run already exists in server. Run id(s): {9010678}
[2018-04-11 20:56:38] Started run 425 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:39] Error in run 418: Run already exists in server. Run id(s): {9010681}
[2018-04-11 20:56:39] Started run 426 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:39] Error in run 419: Run already exists in server. Run id(s): {9010687}
[2018-04-11 20:56:39] Error in run 420: Run already exists in server. Run id(s): {9010691}
[2018-04-11 20:56:39] Started run 427 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:39] Started run 428 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:40] Error in run 421: Run already exists in server. Run id(s): {9010712}
[2018-04-11 20:56:40] Error in run 422: Run already exists in server. Run id(s): {9010716}
[2018-04-11 20:56:40] Error in run 423: Run already exists in server. Run id(s): {9010718}
[2018-04-11 20:56:40] Started run 429 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:40] Started run 431 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:40] Started run 430 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:41] Error in run 424: Run already exists in server. Run id(s): {9010719}
[2018-04-11 20:56:41] Started run 432 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:42] Error in run 425: Run already exists in server. Run id(s): {9010741}
[2018-04-11 20:56:42] Started run 433 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:43] Error in run 426: Run already exists in server. Run id(s): {9010739}
[2018-04-11 20:56:43] Started run 434 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:44] Error in run 428: Run already exists in server. Run id(s): {9010750}
[2018-04-11 20:56:44] Error in run 427: Run already exists in server. Run id(s): {9010745}
[2018-04-11 20:56:44] Started run 435 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:44] Started run 436 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:45] Error in run 429: Run already exists in server. Run id(s): {9010778}
[2018-04-11 20:56:45] Error in run 431: Run already exists in server. Run id(s): {9010779}
[2018-04-11 20:56:45] Started run 437 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:45] Error in run 430: Run already exists in server. Run id(s): {9010777}
[2018-04-11 20:56:45] Started run 438 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:45] Started run 439 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:46] Error in run 432: Run already exists in server. Run id(s): {9010783}
[2018-04-11 20:56:46] Started run 440 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:47] Error in run 433: Run already exists in server. Run id(s): {9015141}
[2018-04-11 20:56:47] Started run 441 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:48] Error in run 434: Run already exists in server. Run id(s): {9015134}
[2018-04-11 20:56:48] Started run 442 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:49] Error in run 435: Run already exists in server. Run id(s): {9015143}
[2018-04-11 20:56:49] Error in run 436: Run already exists in server. Run id(s): {9015279}
[2018-04-11 20:56:49] Started run 443 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:49] Started run 444 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:50] Error in run 437: Run already exists in server. Run id(s): {9015147}
[2018-04-11 20:56:50] Error in run 439: Run already exists in server. Run id(s): {9015185}
[2018-04-11 20:56:50] Error in run 438: Run already exists in server. Run id(s): {9015153}
[2018-04-11 20:56:50] Started run 445 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:50] Started run 446 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:50] Started run 447 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:50] Error in run 440: Run already exists in server. Run id(s): {9015192}
[2018-04-11 20:56:50] Started run 448 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:52] Error in run 441: Run already exists in server. Run id(s): {9015202}
[2018-04-11 20:56:52] Started run 449 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:53] Error in run 442: Run already exists in server. Run id(s): {9015206}
[2018-04-11 20:56:53] Started run 450 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:54] Error in run 444: Run already exists in server. Run id(s): {9015319}
[2018-04-11 20:56:54] Error in run 443: Run already exists in server. Run id(s): {9015212}
[2018-04-11 20:56:54] Started run 451 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:54] Started run 452 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:54] Error in run 445: Run already exists in server. Run id(s): {9015220}
[2018-04-11 20:56:54] Error in run 446: Run already exists in server. Run id(s): {9015256}
[2018-04-11 20:56:54] Started run 453 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:55] Error in run 447: Run already exists in server. Run id(s): {9015338}
[2018-04-11 20:56:54] Started run 454 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:55] Started run 455 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:55] Error in run 448: Run already exists in server. Run id(s): {9015272}
[2018-04-11 20:56:55] Started run 456 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:57] Error in run 449: Run already exists in server. Run id(s): {9015342}
[2018-04-11 20:56:57] Started run 457 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:58] Error in run 450: Run already exists in server. Run id(s): {9015278}
[2018-04-11 20:56:58] Started run 458 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:58] Error in run 451: Run already exists in server. Run id(s): {9015289}
[2018-04-11 20:56:58] Error in run 452: Run already exists in server. Run id(s): {9015332}
[2018-04-11 20:56:58] Started run 459 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:58] Started run 460 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:59] Error in run 454: Run already exists in server. Run id(s): {9015346}
[2018-04-11 20:56:59] Error in run 455: Run already exists in server. Run id(s): {9015344}
[2018-04-11 20:56:59] Error in run 453: Run already exists in server. Run id(s): {9015349}
[2018-04-11 20:56:59] Started run 461 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:59] Started run 462 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:56:59] Started run 463 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:00] Error in run 456: Run already exists in server. Run id(s): {9015357}
[2018-04-11 20:57:00] Started run 464 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:01] Error in run 457: Run already exists in server. Run id(s): {9015452}
[2018-04-11 20:57:01] Started run 465 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:02] Error in run 458: Run already exists in server. Run id(s): {9015397}
[2018-04-11 20:57:02] Started run 466 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:03] Error in run 459: Run already exists in server. Run id(s): {9015407}
[2018-04-11 20:57:03] Error in run 460: Run already exists in server. Run id(s): {9015411}
[2018-04-11 20:57:03] Started run 467 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:03] Started run 468 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:04] Error in run 461: Run already exists in server. Run id(s): {9015426}
[2018-04-11 20:57:04] Error in run 462: Run already exists in server. Run id(s): {9015414}
[2018-04-11 20:57:04] Started run 469 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:04] Error in run 463: Run already exists in server. Run id(s): {9015420, 9015493}
[2018-04-11 20:57:04] Started run 470 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:04] Started run 471 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:05] Error in run 464: Run already exists in server. Run id(s): {9015448}
[2018-04-11 20:57:05] Started run 472 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:06] Error in run 465: Run already exists in server. Run id(s): {9015464}
[2018-04-11 20:57:06] Started run 473 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:07] Error in run 466: Run already exists in server. Run id(s): {9015484}
[2018-04-11 20:57:07] Started run 474 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:08] Error in run 467: Run already exists in server. Run id(s): {9015478}
[2018-04-11 20:57:08] Error in run 468: Run already exists in server. Run id(s): {9015549}
[2018-04-11 20:57:08] Started run 475 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:08] Started run 476 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:09] Error in run 470: Run already exists in server. Run id(s): {9015542}
[2018-04-11 20:57:09] Error in run 469: Run already exists in server. Run id(s): {9015557}
[2018-04-11 20:57:09] Error in run 471: Run already exists in server. Run id(s): {9015650}
[2018-04-11 20:57:09] Started run 477 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:09] Started run 478 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:09] Started run 479 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:09] Error in run 472: Run already exists in server. Run id(s): {9015533}
[2018-04-11 20:57:09] Started run 480 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:11] Error in run 473: Run already exists in server. Run id(s): {9015544}
[2018-04-11 20:57:11] Started run 481 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:12] Error in run 474: Run already exists in server. Run id(s): {9015548}
[2018-04-11 20:57:12] Started run 482 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:13] Error in run 476: Run already exists in server. Run id(s): {9015605}
[2018-04-11 20:57:13] Error in run 475: Run already exists in server. Run id(s): {9015555}
[2018-04-11 20:57:13] Started run 483 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:13] Started run 484 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:13] Error in run 477: Run already exists in server. Run id(s): {9015600}
[2018-04-11 20:57:13] Error in run 478: Run already exists in server. Run id(s): {9015612}
[2018-04-11 20:57:13] Started run 485 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:13] Error in run 479: Run already exists in server. Run id(s): {9015702}
[2018-04-11 20:57:13] Started run 486 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:13] Started run 487 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:14] Error in run 480: Run already exists in server. Run id(s): {9015617}
[2018-04-11 20:57:14] Started run 488 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:16] Error in run 481: Run already exists in server. Run id(s): {9015623}
[2018-04-11 20:57:16] Started run 489 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:17] Error in run 482: Run already exists in server. Run id(s): {9015666}
[2018-04-11 20:57:17] Started run 490 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:17] Error in run 483: Run already exists in server. Run id(s): {9015664}
[2018-04-11 20:57:17] Error in run 484: Run already exists in server. Run id(s): {9015672}
[2018-04-11 20:57:17] Started run 491 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:17] Started run 492 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:18] Error in run 485: Run already exists in server. Run id(s): {9015679}
[2018-04-11 20:57:18] Error in run 486: Run already exists in server. Run id(s): {9015684}
[2018-04-11 20:57:18] Error in run 487: Run already exists in server. Run id(s): {9015689}
[2018-04-11 20:57:18] Started run 493 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:18] Started run 494 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:18] Started run 495 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:19] Error in run 488: Run already exists in server. Run id(s): {9015716}
[2018-04-11 20:57:19] Started run 496 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:20] Error in run 489: Run already exists in server. Run id(s): {9015797}
[2018-04-11 20:57:20] Started run 497 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:21] Error in run 490: Run already exists in server. Run id(s): {9015732}
[2018-04-11 20:57:21] Started run 498 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:22] Error in run 492: Run already exists in server. Run id(s): {9015745}
[2018-04-11 20:57:22] Error in run 491: Run already exists in server. Run id(s): {9015742}
[2018-04-11 20:57:22] Started run 499 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:22] Started run 500 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:23] Error in run 493: Run already exists in server. Run id(s): {9015753}
[2018-04-11 20:57:23] Error in run 494: Run already exists in server. Run id(s): {9015761}
[2018-04-11 20:57:23] Started run 501 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:23] Error in run 495: Run already exists in server. Run id(s): {9015772}
[2018-04-11 20:57:23] Started run 502 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:23] Error in run 496: Run already exists in server. Run id(s): {9015784}
[2018-04-11 20:57:23] Started run 503 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:23] Started run 504 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:25] Error in run 497: Run already exists in server. Run id(s): {9015805}
[2018-04-11 20:57:25] Started run 505 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:26] Error in run 498: Run already exists in server. Run id(s): {9015810}
[2018-04-11 20:57:26] Started run 506 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:27] Error in run 499: Run already exists in server. Run id(s): {9015818}
[2018-04-11 20:57:27] Error in run 500: Run already exists in server. Run id(s): {9015820}
[2018-04-11 20:57:27] Started run 507 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:27] Started run 508 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:28] Error in run 502: Run already exists in server. Run id(s): {9015964}
[2018-04-11 20:57:28] Error in run 501: Run already exists in server. Run id(s): {9015873}
[2018-04-11 20:57:28] Error in run 503: Run already exists in server. Run id(s): {9015851}
[2018-04-11 20:57:28] Started run 509 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:28] Error in run 504: Run already exists in server. Run id(s): {9015864}
[2018-04-11 20:57:28] Started run 510 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:28] Started run 512 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:28] Started run 511 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:30] Error in run 505: Run already exists in server. Run id(s): {9015957}
[2018-04-11 20:57:30] Started run 513 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:31] Error in run 506: Run already exists in server. Run id(s): {9015878}
[2018-04-11 20:57:31] Started run 514 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:32] Error in run 508: Run already exists in server. Run id(s): {9015880}
[2018-04-11 20:57:32] Error in run 507: Run already exists in server. Run id(s): {9015934}
[2018-04-11 20:57:32] Started run 515 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:32] Started run 516 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:32] Error in run 509: Run already exists in server. Run id(s): {9015926}
[2018-04-11 20:57:32] Error in run 510: Run already exists in server. Run id(s): {9015940}
[2018-04-11 20:57:32] Started run 517 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:32] Error in run 512: Run already exists in server. Run id(s): {9015945}
[2018-04-11 20:57:32] Started run 518 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:33] Error in run 511: Run already exists in server. Run id(s): {9015939}
[2018-04-11 20:57:32] Started run 519 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:33] Started run 520 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:35] Error in run 513: Run already exists in server. Run id(s): {9015950}
[2018-04-11 20:57:35] Started run 521 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:35] Error in run 514: Run already exists in server. Run id(s): {9015984}
[2018-04-11 20:57:35] Started run 522 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:36] Error in run 515: Run already exists in server. Run id(s): {9015999}
[2018-04-11 20:57:36] Error in run 516: Run already exists in server. Run id(s): {9016037}
[2018-04-11 20:57:36] Started run 523 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:36] Started run 524 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:37] Error in run 517: Run already exists in server. Run id(s): {9016072}
[2018-04-11 20:57:37] Error in run 519: Run already exists in server. Run id(s): {9016028}
[2018-04-11 20:57:37] Error in run 518: Run already exists in server. Run id(s): {9016065}
[2018-04-11 20:57:37] Started run 525 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:37] Error in run 520: Run already exists in server. Run id(s): {9016032}
[2018-04-11 20:57:37] Started run 528 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:37] Started run 526 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:37] Started run 527 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:39] Error in run 521: Run already exists in server. Run id(s): {9016036}
[2018-04-11 20:57:39] Started run 529 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:40] Error in run 522: Run already exists in server. Run id(s): {9016058}
[2018-04-11 20:57:40] Started run 530 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:41] Error in run 523: Run already exists in server. Run id(s): {9016201}
[2018-04-11 20:57:41] Error in run 524: Run already exists in server. Run id(s): {9016096}
[2018-04-11 20:57:41] Started run 531 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:41] Started run 532 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:42] Error in run 525: Run already exists in server. Run id(s): {9016143}
[2018-04-11 20:57:42] Error in run 528: Run already exists in server. Run id(s): {9016129}
[2018-04-11 20:57:42] Started run 533 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:42] Error in run 526: Run already exists in server. Run id(s): {9016099}
[2018-04-11 20:57:42] Started run 534 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:42] Started run 535 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:42] Error in run 527: Run already exists in server. Run id(s): {9016319}
[2018-04-11 20:57:42] Started run 536 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:44] Error in run 529: Run already exists in server. Run id(s): {9016137}
[2018-04-11 20:57:44] Started run 537 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:45] Error in run 530: Run already exists in server. Run id(s): {9016141}
[2018-04-11 20:57:45] Started run 538 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:46] Error in run 531: Run already exists in server. Run id(s): {9016158}
[2018-04-11 20:57:46] Started run 539 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:46] Error in run 532: Run already exists in server. Run id(s): {9016168}
[2018-04-11 20:57:46] Started run 540 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:46] Error in run 533: Run already exists in server. Run id(s): {9016195}
[2018-04-11 20:57:46] Started run 541 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:47] Error in run 534: Run already exists in server. Run id(s): {9016191}
[2018-04-11 20:57:47] Error in run 535: Run already exists in server. Run id(s): {9016213}
[2018-04-11 20:57:47] Started run 542 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:47] Error in run 536: Run already exists in server. Run id(s): {9016210}
[2018-04-11 20:57:47] Started run 543 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:47] Started run 544 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:49] Error in run 537: Run already exists in server. Run id(s): {9016223}
[2018-04-11 20:57:49] Started run 545 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:50] Error in run 538: Run already exists in server. Run id(s): {9016236}
[2018-04-11 20:57:50] Started run 546 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:50] Error in run 539: Run already exists in server. Run id(s): {9016262}
[2018-04-11 20:57:50] Started run 547 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:51] Error in run 540: Run already exists in server. Run id(s): {9016268}
[2018-04-11 20:57:51] Started run 548 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:51] Error in run 541: Run already exists in server. Run id(s): {9016284}
[2018-04-11 20:57:51] Started run 549 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:51] Error in run 542: Run already exists in server. Run id(s): {9016273}
[2018-04-11 20:57:51] Error in run 543: Run already exists in server. Run id(s): {9016279}
[2018-04-11 20:57:51] Started run 550 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:51] Error in run 544: Run already exists in server. Run id(s): {9016293}
[2018-04-11 20:57:51] Started run 551 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:51] Started run 552 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:54] Error in run 545: Run already exists in server. Run id(s): {9016311}
[2018-04-11 20:57:54] Started run 553 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:54] Error in run 546: Run already exists in server. Run id(s): {9016365}
[2018-04-11 20:57:54] Started run 554 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:55] Error in run 547: Run already exists in server. Run id(s): {9016341}
[2018-04-11 20:57:55] Started run 555 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:55] Error in run 548: Run already exists in server. Run id(s): {9016338}
[2018-04-11 20:57:55] Started run 556 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:56] Error in run 549: Run already exists in server. Run id(s): {9016345}
[2018-04-11 20:57:56] Started run 557 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:56] Error in run 551: Run already exists in server. Run id(s): {9016394}
[2018-04-11 20:57:56] Error in run 550: Run already exists in server. Run id(s): {9016352}
[2018-04-11 20:57:56] Error in run 552: Run already exists in server. Run id(s): {9016375}
[2018-04-11 20:57:56] Started run 558 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:56] Started run 559 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:56] Started run 560 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:58] Error in run 553: Run already exists in server. Run id(s): {9016380}
[2018-04-11 20:57:58] Started run 561 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:57:59] Error in run 554: Run already exists in server. Run id(s): {9016403}
[2018-04-11 20:57:59] Started run 562 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:00] Error in run 555: Run already exists in server. Run id(s): {9016458}
[2018-04-11 20:58:00] Started run 563 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:00] Error in run 556: Run already exists in server. Run id(s): {9016411}
[2018-04-11 20:58:00] Started run 564 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:00] Error in run 557: Run already exists in server. Run id(s): {9016420}
[2018-04-11 20:58:00] Started run 565 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:01] Error in run 558: Run already exists in server. Run id(s): {9016426}
[2018-04-11 20:58:01] Error in run 559: Run already exists in server. Run id(s): {9016440}
[2018-04-11 20:58:01] Started run 566 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:01] Started run 567 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:01] Error in run 560: Run already exists in server. Run id(s): {9016447}
[2018-04-11 20:58:01] Started run 568 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:03] Error in run 561: Run already exists in server. Run id(s): {9016469}
[2018-04-11 20:58:03] Started run 569 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:04] Error in run 562: Run already exists in server. Run id(s): {9016475}
[2018-04-11 20:58:04] Started run 570 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:04] Error in run 563: Run already exists in server. Run id(s): {9016484}
[2018-04-11 20:58:04] Started run 571 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:05] Error in run 564: Run already exists in server. Run id(s): {9016561}
[2018-04-11 20:58:05] Started run 572 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:05] Error in run 565: Run already exists in server. Run id(s): {9016503}
[2018-04-11 20:58:05] Started run 573 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:06] Error in run 568: Run already exists in server. Run id(s): {9016525}
[2018-04-11 20:58:06] Error in run 566: Run already exists in server. Run id(s): {9016545}
[2018-04-11 20:58:06] Error in run 567: Run already exists in server. Run id(s): {9016530}
[2018-04-11 20:58:06] Started run 574 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:06] Started run 575 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:06] Started run 576 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:08] Error in run 569: Run already exists in server. Run id(s): {9016540}
[2018-04-11 20:58:08] Started run 577 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:09] Error in run 570: Run already exists in server. Run id(s): {9016532}
[2018-04-11 20:58:09] Started run 578 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:09] Error in run 571: Run already exists in server. Run id(s): {9016626}
[2018-04-11 20:58:09] Started run 579 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:10] Error in run 573: Run already exists in server. Run id(s): {9016614}
[2018-04-11 20:58:10] Error in run 572: Run already exists in server. Run id(s): {9016567}
[2018-04-11 20:58:10] Started run 580 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:10] Started run 581 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:10] Error in run 574: Run already exists in server. Run id(s): {9016598}
[2018-04-11 20:58:10] Error in run 575: Run already exists in server. Run id(s): {9016594}
[2018-04-11 20:58:10] Started run 582 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:10] Error in run 576: Run already exists in server. Run id(s): {9016603}
[2018-04-11 20:58:10] Started run 583 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:10] Started run 584 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:13] Error in run 577: Run already exists in server. Run id(s): {9016611}
[2018-04-11 20:58:13] Started run 585 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:13] Error in run 578: Run already exists in server. Run id(s): {9016623}
[2018-04-11 20:58:13] Started run 586 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:13] Error in run 579: Run already exists in server. Run id(s): {9016630}
[2018-04-11 20:58:13] Started run 587 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:14] Error in run 580: Run already exists in server. Run id(s): {9016668}
[2018-04-11 20:58:14] Error in run 581: Run already exists in server. Run id(s): {9016659}
[2018-04-11 20:58:14] Started run 588 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:14] Started run 589 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:15] Error in run 582: Run already exists in server. Run id(s): {9016664}
[2018-04-11 20:58:15] Error in run 583: Run already exists in server. Run id(s): {9016674}
[2018-04-11 20:58:15] Error in run 584: Run already exists in server. Run id(s): {9016677}
[2018-04-11 20:58:15] Started run 590 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:15] Started run 591 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:15] Started run 592 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:17] Error in run 585: Run already exists in server. Run id(s): {9016716}
[2018-04-11 20:58:17] Started run 593 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:18] Error in run 586: Run already exists in server. Run id(s): {9016698}
[2018-04-11 20:58:18] Started run 594 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:18] Error in run 587: Run already exists in server. Run id(s): {9016695}
[2018-04-11 20:58:18] Started run 595 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:19] Error in run 588: Run already exists in server. Run id(s): {9016723}
[2018-04-11 20:58:19] Error in run 589: Run already exists in server. Run id(s): {9016731}
[2018-04-11 20:58:19] Started run 596 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:19] Started run 597 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:20] Error in run 590: Run already exists in server. Run id(s): {9016738}
[2018-04-11 20:58:20] Error in run 591: Run already exists in server. Run id(s): {9016746}
[2018-04-11 20:58:20] Started run 598 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:20] Started run 599 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:20] Error in run 592: Run already exists in server. Run id(s): {9016751}
[2018-04-11 20:58:20] Started run 600 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:22] Error in run 593: Run already exists in server. Run id(s): {9016765}
[2018-04-11 20:58:22] Started run 601 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:22] Error in run 594: Run already exists in server. Run id(s): {9016767}
[2018-04-11 20:58:22] Started run 602 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:23] Error in run 595: Run already exists in server. Run id(s): {9016788}
[2018-04-11 20:58:23] Started run 603 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:24] Error in run 596: Run already exists in server. Run id(s): {9016824}
[2018-04-11 20:58:24] Error in run 597: Run already exists in server. Run id(s): {9016799}
[2018-04-11 20:58:24] Started run 604 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:24] Started run 605 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:25] Error in run 600: Run already exists in server. Run id(s): {9016815}
[2018-04-11 20:58:25] Error in run 599: Run already exists in server. Run id(s): {9016805}
[2018-04-11 20:58:25] Error in run 598: Run already exists in server. Run id(s): {9016864, 9016795}
[2018-04-11 20:58:25] Started run 606 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:25] Started run 607 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:25] Started run 608 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:27] Error in run 601: Run already exists in server. Run id(s): {9016862}
[2018-04-11 20:58:27] Started run 609 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:27] Error in run 602: Run already exists in server. Run id(s): {9016836}
[2018-04-11 20:58:27] Started run 610 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:28] Error in run 603: Run already exists in server. Run id(s): {9016854}
[2018-04-11 20:58:28] Started run 611 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:28] Error in run 605: Run already exists in server. Run id(s): {9016871}
[2018-04-11 20:58:29] Error in run 604: Run already exists in server. Run id(s): {9016866}
[2018-04-11 20:58:28] Started run 612 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:29] Started run 613 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:29] Error in run 606: Run already exists in server. Run id(s): {9016881}
[2018-04-11 20:58:29] Error in run 607: Run already exists in server. Run id(s): {9016897}
[2018-04-11 20:58:29] Started run 614 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:29] Error in run 608: Run already exists in server. Run id(s): {9016950}
[2018-04-11 20:58:29] Started run 615 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:29] Started run 616 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:32] Error in run 609: Run already exists in server. Run id(s): {9016954}
[2018-04-11 20:58:32] Started run 617 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:32] Error in run 610: Run already exists in server. Run id(s): {9016948}
[2018-04-11 20:58:32] Started run 618 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:32] Error in run 611: Run already exists in server. Run id(s): {9016923}
[2018-04-11 20:58:32] Started run 619 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:33] Error in run 612: Run already exists in server. Run id(s): {9016929}
[2018-04-11 20:58:33] Error in run 613: Run already exists in server. Run id(s): {9016968}
[2018-04-11 20:58:33] Started run 620 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:33] Started run 621 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:34] Error in run 615: Run already exists in server. Run id(s): {9016962}
[2018-04-11 20:58:34] Error in run 614: Run already exists in server. Run id(s): {9016946}
[2018-04-11 20:58:34] Error in run 616: Run already exists in server. Run id(s): {9016979}
[2018-04-11 20:58:34] Started run 622 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:34] Started run 623 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:34] Started run 624 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:36] Error in run 617: Run already exists in server. Run id(s): {9016987}
[2018-04-11 20:58:36] Error in run 618: Run already exists in server. Run id(s): {9017008}
[2018-04-11 20:58:36] Started run 625 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:36] Started run 626 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:37] Error in run 619: Run already exists in server. Run id(s): {9017015}
[2018-04-11 20:58:37] Started run 627 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:38] Error in run 621: Run already exists in server. Run id(s): {9017018}
[2018-04-11 20:58:38] Error in run 620: Run already exists in server. Run id(s): {9017020}
[2018-04-11 20:58:38] Started run 628 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:38] Started run 629 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:39] Error in run 622: Run already exists in server. Run id(s): {9017024}
[2018-04-11 20:58:39] Error in run 623: Run already exists in server. Run id(s): {9017031}
[2018-04-11 20:58:39] Started run 630 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:39] Error in run 624: Run already exists in server. Run id(s): {9017048}
[2018-04-11 20:58:39] Started run 631 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:39] Started run 632 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:41] Error in run 625: Run already exists in server. Run id(s): {9017057}
[2018-04-11 20:58:41] Error in run 626: Run already exists in server. Run id(s): {9017095}
[2018-04-11 20:58:41] Started run 633 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:41] Started run 634 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:42] Error in run 627: Run already exists in server. Run id(s): {9017079}
[2018-04-11 20:58:42] Started run 635 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:43] Error in run 628: Run already exists in server. Run id(s): {9017089}
[2018-04-11 20:58:43] Error in run 629: Run already exists in server. Run id(s): {9017087}
[2018-04-11 20:58:43] Started run 636 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:43] Started run 637 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:44] Error in run 631: Run already exists in server. Run id(s): {9017103}
[2018-04-11 20:58:44] Error in run 630: Run already exists in server. Run id(s): {9017096}
[2018-04-11 20:58:44] Error in run 632: Run already exists in server. Run id(s): {9017109}
[2018-04-11 20:58:44] Started run 638 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:44] Started run 639 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:44] Started run 640 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:46] Error in run 633: Run already exists in server. Run id(s): {9017135}
[2018-04-11 20:58:46] Error in run 634: Run already exists in server. Run id(s): {9017146}
[2018-04-11 20:58:46] Started run 641 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:46] Started run 642 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:47] Error in run 635: Run already exists in server. Run id(s): {9017283}
[2018-04-11 20:58:47] Started run 643 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:47] Error in run 636: Run already exists in server. Run id(s): {9017168}
[2018-04-11 20:58:47] Error in run 637: Run already exists in server. Run id(s): {9017215}
[2018-04-11 20:58:47] Started run 644 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:47] Started run 645 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:48] Error in run 638: Run already exists in server. Run id(s): {9017163}
[2018-04-11 20:58:48] Error in run 639: Run already exists in server. Run id(s): {9017254}
[2018-04-11 20:58:48] Started run 646 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:48] Error in run 640: Run already exists in server. Run id(s): {9017174}
[2018-04-11 20:58:48] Started run 647 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:48] Started run 648 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:51] Error in run 642: Run already exists in server. Run id(s): {9017207}
[2018-04-11 20:58:51] Error in run 641: Run already exists in server. Run id(s): {9017286}
[2018-04-11 20:58:51] Started run 649 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:51] Started run 650 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:51] Error in run 643: Run already exists in server. Run id(s): {9017232}
[2018-04-11 20:58:51] Started run 651 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:52] Error in run 644: Run already exists in server. Run id(s): {9017241}
[2018-04-11 20:58:52] Error in run 645: Run already exists in server. Run id(s): {9017245}
[2018-04-11 20:58:52] Started run 652 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:52] Started run 653 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:53] Error in run 647: Run already exists in server. Run id(s): {9017374}
[2018-04-11 20:58:53] Error in run 646: Run already exists in server. Run id(s): {9017275}
[2018-04-11 20:58:53] Error in run 648: Run already exists in server. Run id(s): {9017306}
[2018-04-11 20:58:53] Started run 655 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:53] Started run 656 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:53] Started run 654 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:55] Error in run 649: Run already exists in server. Run id(s): {9017311}
[2018-04-11 20:58:55] Error in run 650: Run already exists in server. Run id(s): {9017317}
[2018-04-11 20:58:55] Started run 657 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:55] Started run 658 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:56] Error in run 651: Run already exists in server. Run id(s): {9017322}
[2018-04-11 20:58:56] Started run 659 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:57] Error in run 653: Run already exists in server. Run id(s): {9017353}
[2018-04-11 20:58:57] Error in run 652: Run already exists in server. Run id(s): {9017348}
[2018-04-11 20:58:57] Started run 660 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:57] Started run 661 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:58] Error in run 655: Run already exists in server. Run id(s): {9017381}
[2018-04-11 20:58:58] Error in run 656: Run already exists in server. Run id(s): {9017371}
[2018-04-11 20:58:58] Started run 662 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:58] Started run 663 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:58:58] Error in run 654: Run already exists in server. Run id(s): {9017359}
[2018-04-11 20:58:58] Started run 664 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:00] Error in run 658: Run already exists in server. Run id(s): {9017385}
[2018-04-11 20:59:00] Error in run 657: Run already exists in server. Run id(s): {9017392}
[2018-04-11 20:59:00] Started run 665 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:00] Started run 666 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:01] Error in run 659: Run already exists in server. Run id(s): {9017419}
[2018-04-11 20:59:01] Started run 667 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:02] Error in run 660: Run already exists in server. Run id(s): {9017519}
[2018-04-11 20:59:02] Error in run 661: Run already exists in server. Run id(s): {9017433}
[2018-04-11 20:59:02] Started run 668 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:02] Started run 669 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:03] Error in run 663: Run already exists in server. Run id(s): {9017449}
[2018-04-11 20:59:03] Error in run 662: Run already exists in server. Run id(s): {9017441}
[2018-04-11 20:59:03] Error in run 664: Run already exists in server. Run id(s): {9017452}
[2018-04-11 20:59:03] Started run 670 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:03] Started run 671 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:03] Started run 672 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:05] Error in run 666: Run already exists in server. Run id(s): {9017461}
[2018-04-11 20:59:05] Error in run 665: Run already exists in server. Run id(s): {9017455}
[2018-04-11 20:59:05] Started run 673 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:05] Started run 674 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:06] Error in run 667: Run already exists in server. Run id(s): {9017487}
[2018-04-11 20:59:06] Started run 675 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:06] Error in run 668: Run already exists in server. Run id(s): {9017501}
[2018-04-11 20:59:06] Error in run 669: Run already exists in server. Run id(s): {9017510}
[2018-04-11 20:59:06] Started run 676 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:06] Started run 677 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:07] Error in run 670: Run already exists in server. Run id(s): {9017516}
[2018-04-11 20:59:07] Error in run 671: Run already exists in server. Run id(s): {9017537}
[2018-04-11 20:59:07] Started run 678 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:07] Error in run 672: Run already exists in server. Run id(s): {9017684}
[2018-04-11 20:59:07] Started run 679 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:07] Started run 680 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:10] Error in run 673: Run already exists in server. Run id(s): {9017530}
[2018-04-11 20:59:10] Error in run 674: Run already exists in server. Run id(s): {9017586}
[2018-04-11 20:59:10] Started run 681 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:10] Started run 682 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:10] Error in run 675: Run already exists in server. Run id(s): {9017627}
[2018-04-11 20:59:10] Started run 683 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:11] Error in run 676: Run already exists in server. Run id(s): {9017776}
[2018-04-11 20:59:11] Error in run 677: Run already exists in server. Run id(s): {9017589}
[2018-04-11 20:59:11] Started run 684 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:11] Started run 685 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:12] Error in run 679: Run already exists in server. Run id(s): {9017601}
[2018-04-11 20:59:12] Error in run 678: Run already exists in server. Run id(s): {9017596, 9017644}
[2018-04-11 20:59:12] Error in run 680: Run already exists in server. Run id(s): {9017609}
[2018-04-11 20:59:12] Started run 686 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:12] Started run 687 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:12] Started run 688 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:14] Error in run 681: Run already exists in server. Run id(s): {9017659}
[2018-04-11 20:59:14] Error in run 682: Run already exists in server. Run id(s): {9017665}
[2018-04-11 20:59:14] Started run 689 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:14] Started run 690 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:15] Error in run 683: Run already exists in server. Run id(s): {9017745}
[2018-04-11 20:59:15] Started run 691 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:16] Error in run 684: Run already exists in server. Run id(s): {9017675}
[2018-04-11 20:59:16] Error in run 685: Run already exists in server. Run id(s): {9017699}
[2018-04-11 20:59:16] Started run 692 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:16] Started run 693 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:17] Error in run 686: Run already exists in server. Run id(s): {9017706}
[2018-04-11 20:59:17] Error in run 687: Run already exists in server. Run id(s): {9017714}
[2018-04-11 20:59:17] Started run 694 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:17] Started run 695 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:17] Error in run 688: Run already exists in server. Run id(s): {9017725}
[2018-04-11 20:59:17] Started run 696 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:19] Error in run 689: Run already exists in server. Run id(s): {9017777}
[2018-04-11 20:59:19] Error in run 690: Run already exists in server. Run id(s): {9017756}
[2018-04-11 20:59:19] Started run 697 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:19] Started run 698 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:20] Error in run 691: Run already exists in server. Run id(s): {9017759}
[2018-04-11 20:59:20] Started run 699 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:21] Error in run 692: Run already exists in server. Run id(s): {9017765}
[2018-04-11 20:59:21] Error in run 693: Run already exists in server. Run id(s): {9017874}
[2018-04-11 20:59:21] Started run 700 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:21] Started run 701 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:21] Error in run 695: Run already exists in server. Run id(s): {9017821}
[2018-04-11 20:59:21] Error in run 694: Run already exists in server. Run id(s): {9017794}
[2018-04-11 20:59:22] Error in run 696: Run already exists in server. Run id(s): {9017833}
[2018-04-11 20:59:21] Started run 702 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:21] Started run 703 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:22] Started run 704 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:24] Error in run 697: Run already exists in server. Run id(s): {9017825}
[2018-04-11 20:59:24] Error in run 698: Run already exists in server. Run id(s): {9017839}
[2018-04-11 20:59:24] Started run 705 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:24] Started run 706 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:25] Error in run 699: Run already exists in server. Run id(s): {9017846}
[2018-04-11 20:59:25] Started run 707 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:25] Error in run 700: Run already exists in server. Run id(s): {9017849}
[2018-04-11 20:59:25] Error in run 701: Run already exists in server. Run id(s): {9017864}
[2018-04-11 20:59:25] Started run 708 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:25] Started run 709 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:26] Error in run 702: Run already exists in server. Run id(s): {9017896}
[2018-04-11 20:59:26] Error in run 703: Run already exists in server. Run id(s): {9017901}
[2018-04-11 20:59:26] Started run 710 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:26] Error in run 704: Run already exists in server. Run id(s): {9017905}
[2018-04-11 20:59:26] Started run 711 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:26] Started run 712 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:29] Error in run 706: Run already exists in server. Run id(s): {9017997}
[2018-04-11 20:59:29] Error in run 705: Run already exists in server. Run id(s): {9017913}
[2018-04-11 20:59:29] Started run 713 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:29] Started run 714 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:29] Error in run 707: Run already exists in server. Run id(s): {9018009}
[2018-04-11 20:59:29] Started run 715 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:30] Error in run 708: Run already exists in server. Run id(s): {9017939}
[2018-04-11 20:59:30] Error in run 709: Run already exists in server. Run id(s): {9018008, 9017935}
[2018-04-11 20:59:30] Started run 716 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:30] Started run 717 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:31] Error in run 710: Run already exists in server. Run id(s): {9017969}
[2018-04-11 20:59:31] Error in run 711: Run already exists in server. Run id(s): {9017971}
[2018-04-11 20:59:31] Started run 718 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:31] Error in run 712: Run already exists in server. Run id(s): {9017981}
[2018-04-11 20:59:31] Started run 719 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:31] Started run 720 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:33] Error in run 713: Run already exists in server. Run id(s): {9018005}
[2018-04-11 20:59:33] Started run 721 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:33] Error in run 714: Run already exists in server. Run id(s): {9018004}
[2018-04-11 20:59:33] Started run 722 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:34] Error in run 715: Run already exists in server. Run id(s): {9018045}
[2018-04-11 20:59:34] Started run 723 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:35] Error in run 717: Run already exists in server. Run id(s): {9018052}
[2018-04-11 20:59:35] Error in run 716: Run already exists in server. Run id(s): {9018048}
[2018-04-11 20:59:35] Started run 724 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:35] Started run 725 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:36] Error in run 718: Run already exists in server. Run id(s): {9018071}
[2018-04-11 20:59:36] Error in run 719: Run already exists in server. Run id(s): {9018128, 9018204}
[2018-04-11 20:59:36] Started run 726 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:36] Error in run 720: Run already exists in server. Run id(s): {9018077}
[2018-04-11 20:59:36] Started run 727 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:36] Started run 728 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:38] Error in run 721: Run already exists in server. Run id(s): {9018090}
[2018-04-11 20:59:38] Error in run 722: Run already exists in server. Run id(s): {9018083}
[2018-04-11 20:59:38] Started run 729 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:38] Started run 730 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:39] Error in run 723: Run already exists in server. Run id(s): {9018107}
[2018-04-11 20:59:39] Started run 731 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:40] Error in run 724: Run already exists in server. Run id(s): {9018118}
[2018-04-11 20:59:40] Error in run 725: Run already exists in server. Run id(s): {9018162}
[2018-04-11 20:59:40] Started run 732 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:40] Started run 733 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:40] Error in run 727: Run already exists in server. Run id(s): {9018274}
[2018-04-11 20:59:40] Error in run 726: Run already exists in server. Run id(s): {9018142}
[2018-04-11 20:59:40] Error in run 728: Run already exists in server. Run id(s): {9018154}
[2018-04-11 20:59:40] Started run 734 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:40] Started run 735 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:40] Started run 736 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:43] Error in run 729: Run already exists in server. Run id(s): {9018160}
[2018-04-11 20:59:43] Started run 737 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:43] Error in run 730: Run already exists in server. Run id(s): {9018170}
[2018-04-11 20:59:43] Started run 738 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:44] Error in run 731: Run already exists in server. Run id(s): {9018203}
[2018-04-11 20:59:44] Started run 739 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:44] Error in run 733: Run already exists in server. Run id(s): {9018212}
[2018-04-11 20:59:44] Error in run 732: Run already exists in server. Run id(s): {9018201}
[2018-04-11 20:59:44] Started run 740 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:44] Started run 741 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:45] Error in run 734: Run already exists in server. Run id(s): {9018210}
[2018-04-11 20:59:45] Error in run 735: Run already exists in server. Run id(s): {9018211}
[2018-04-11 20:59:45] Started run 742 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:45] Error in run 736: Run already exists in server. Run id(s): {9018224}
[2018-04-11 20:59:45] Started run 743 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:45] Started run 744 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:48] Error in run 737: Run already exists in server. Run id(s): {9018290}
[2018-04-11 20:59:48] Error in run 738: Run already exists in server. Run id(s): {9018262}
[2018-04-11 20:59:48] Started run 745 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:48] Started run 746 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:48] Error in run 739: Run already exists in server. Run id(s): {9018288}
[2018-04-11 20:59:48] Started run 747 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:49] Error in run 740: Run already exists in server. Run id(s): {9018260}
[2018-04-11 20:59:49] Started run 748 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:49] Error in run 741: Run already exists in server. Run id(s): {9018309}
[2018-04-11 20:59:49] Started run 749 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:50] Error in run 742: Run already exists in server. Run id(s): {9018307}
[2018-04-11 20:59:50] Error in run 744: Run already exists in server. Run id(s): {9018330}
[2018-04-11 20:59:50] Started run 750 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:50] Error in run 743: Run already exists in server. Run id(s): {9018286}
[2018-04-11 20:59:50] Started run 752 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:50] Started run 751 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:52] Error in run 745: Run already exists in server. Run id(s): {9018529}
[2018-04-11 20:59:52] Error in run 746: Run already exists in server. Run id(s): {9018326}
[2018-04-11 20:59:52] Started run 753 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:52] Started run 754 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:53] Error in run 747: Run already exists in server. Run id(s): {9018363}
[2018-04-11 20:59:53] Started run 755 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:54] Error in run 749: Run already exists in server. Run id(s): {9018364}
[2018-04-11 20:59:54] Error in run 748: Run already exists in server. Run id(s): {9018369}
[2018-04-11 20:59:54] Started run 756 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:54] Started run 757 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:55] Error in run 750: Run already exists in server. Run id(s): {9018384}
[2018-04-11 20:59:55] Started run 758 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:55] Error in run 752: Run already exists in server. Run id(s): {9018400}
[2018-04-11 20:59:55] Error in run 751: Run already exists in server. Run id(s): {9018417}
[2018-04-11 20:59:55] Started run 759 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:55] Started run 760 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:57] Error in run 753: Run already exists in server. Run id(s): {9018405}
[2018-04-11 20:59:57] Error in run 754: Run already exists in server. Run id(s): {9018446}
[2018-04-11 20:59:57] Started run 761 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:57] Started run 762 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:58] Error in run 755: Run already exists in server. Run id(s): {9018442}
[2018-04-11 20:59:58] Started run 763 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:59] Error in run 756: Run already exists in server. Run id(s): {9018534}
[2018-04-11 20:59:59] Error in run 757: Run already exists in server. Run id(s): {9018456}
[2018-04-11 20:59:59] Started run 764 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:59] Started run 765 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:59] Error in run 758: Run already exists in server. Run id(s): {9018475}
[2018-04-11 20:59:59] Error in run 760: Run already exists in server. Run id(s): {9018485}
[2018-04-11 20:59:59] Error in run 759: Run already exists in server. Run id(s): {9018498}
[2018-04-11 20:59:59] Started run 766 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:59] Started run 768 on task 9981, dataset 'cnae-9'.
[2018-04-11 20:59:59] Started run 767 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:02] Error in run 761: Run already exists in server. Run id(s): {9018594}
[2018-04-11 21:00:02] Error in run 762: Run already exists in server. Run id(s): {9018519}
[2018-04-11 21:00:02] Started run 769 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:02] Started run 770 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:03] Error in run 763: Run already exists in server. Run id(s): {9018524}
[2018-04-11 21:00:03] Started run 771 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:03] Error in run 765: Run already exists in server. Run id(s): {9018552}
[2018-04-11 21:00:03] Error in run 764: Run already exists in server. Run id(s): {9018603}
[2018-04-11 21:00:03] Started run 772 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:03] Started run 773 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:04] Error in run 766: Run already exists in server. Run id(s): {9018637}
[2018-04-11 21:00:04] Error in run 768: Run already exists in server. Run id(s): {9018592}
[2018-04-11 21:00:04] Started run 774 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:04] Error in run 767: Run already exists in server. Run id(s): {9018735}
[2018-04-11 21:00:04] Started run 775 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:04] Started run 776 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:06] Error in run 770: Run already exists in server. Run id(s): {9018605}
[2018-04-11 21:00:06] Error in run 769: Run already exists in server. Run id(s): {9018759}
[2018-04-11 21:00:06] Started run 777 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:06] Started run 778 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:07] Error in run 771: Run already exists in server. Run id(s): {9018621}
[2018-04-11 21:00:07] Started run 779 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:08] Error in run 772: Run already exists in server. Run id(s): {9018657}
[2018-04-11 21:00:08] Error in run 773: Run already exists in server. Run id(s): {9018666}
[2018-04-11 21:00:08] Started run 780 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:08] Started run 781 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:09] Error in run 776: Run already exists in server. Run id(s): {9018702}
[2018-04-11 21:00:09] Error in run 774: Run already exists in server. Run id(s): {9018680}
[2018-04-11 21:00:09] Error in run 775: Run already exists in server. Run id(s): {9018808}
[2018-04-11 21:00:09] Started run 782 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:09] Started run 783 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:09] Started run 784 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:11] Error in run 777: Run already exists in server. Run id(s): {9018765}
[2018-04-11 21:00:11] Error in run 778: Run already exists in server. Run id(s): {9018738}
[2018-04-11 21:00:11] Started run 785 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:11] Started run 786 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:12] Error in run 779: Run already exists in server. Run id(s): {9018731}
[2018-04-11 21:00:12] Started run 787 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:13] Error in run 781: Run already exists in server. Run id(s): {9018777}
[2018-04-11 21:00:13] Error in run 780: Run already exists in server. Run id(s): {9018801}
[2018-04-11 21:00:13] Started run 788 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:13] Started run 789 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:14] Error in run 782: Run already exists in server. Run id(s): {9018825}
[2018-04-11 21:00:14] Error in run 783: Run already exists in server. Run id(s): {9018807}
[2018-04-11 21:00:14] Started run 790 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:14] Started run 791 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:14] Error in run 784: Run already exists in server. Run id(s): {9018813}
[2018-04-11 21:00:14] Started run 792 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:16] Error in run 785: Run already exists in server. Run id(s): {9018853}
[2018-04-11 21:00:16] Error in run 786: Run already exists in server. Run id(s): {9018839}
[2018-04-11 21:00:16] Started run 793 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:16] Started run 794 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:17] Error in run 787: Run already exists in server. Run id(s): {9018866}
[2018-04-11 21:00:17] Started run 795 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:17] Error in run 788: Run already exists in server. Run id(s): {9018864}
[2018-04-11 21:00:17] Error in run 789: Run already exists in server. Run id(s): {9018882}
[2018-04-11 21:00:17] Started run 796 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:17] Started run 797 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:18] Error in run 792: Run already exists in server. Run id(s): {9018899}
[2018-04-11 21:00:18] Error in run 790: Run already exists in server. Run id(s): {9018879}
[2018-04-11 21:00:18] Error in run 791: Run already exists in server. Run id(s): {9018888}
[2018-04-11 21:00:18] Started run 798 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:18] Started run 799 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:18] Started run 800 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:21] Error in run 793: Run already exists in server. Run id(s): {9018912}
[2018-04-11 21:00:21] Error in run 794: Run already exists in server. Run id(s): {9019090}
[2018-04-11 21:00:21] Started run 801 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:21] Started run 802 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:22] Error in run 795: Run already exists in server. Run id(s): {9018944}
[2018-04-11 21:00:22] Started run 803 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:22] Error in run 797: Run already exists in server. Run id(s): {9018956}
[2018-04-11 21:00:22] Error in run 796: Run already exists in server. Run id(s): {9018940}
[2018-04-11 21:00:22] Started run 804 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:22] Started run 805 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:23] Error in run 798: Run already exists in server. Run id(s): {9018961}
[2018-04-11 21:00:23] Started run 806 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:23] Error in run 799: Run already exists in server. Run id(s): {9018964}
[2018-04-11 21:00:23] Started run 807 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:23] Error in run 800: Run already exists in server. Run id(s): {9018969}
[2018-04-11 21:00:23] Started run 808 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:25] Error in run 801: Run already exists in server. Run id(s): {9018976}
[2018-04-11 21:00:25] Error in run 802: Run already exists in server. Run id(s): {9019015}
[2018-04-11 21:00:25] Started run 809 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:25] Started run 810 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:26] Error in run 803: Run already exists in server. Run id(s): {9019008}
[2018-04-11 21:00:26] Started run 811 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:27] Error in run 804: Run already exists in server. Run id(s): {9019020}
[2018-04-11 21:00:27] Error in run 805: Run already exists in server. Run id(s): {9019148}
[2018-04-11 21:00:27] Started run 812 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:27] Started run 813 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:28] Error in run 807: Run already exists in server. Run id(s): {9019038}
[2018-04-11 21:00:28] Error in run 806: Run already exists in server. Run id(s): {9019026}
[2018-04-11 21:00:28] Started run 814 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:28] Error in run 808: Run already exists in server. Run id(s): {9019049}
[2018-04-11 21:00:28] Started run 815 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:28] Started run 816 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:30] Error in run 809: Run already exists in server. Run id(s): {9019087}
[2018-04-11 21:00:30] Started run 817 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:30] Error in run 810: Run already exists in server. Run id(s): {9019105}
[2018-04-11 21:00:30] Started run 818 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:31] Error in run 811: Run already exists in server. Run id(s): {9019094}
[2018-04-11 21:00:31] Started run 819 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:32] Error in run 812: Run already exists in server. Run id(s): {9019098}
[2018-04-11 21:00:32] Error in run 813: Run already exists in server. Run id(s): {9019109}
[2018-04-11 21:00:32] Started run 820 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:32] Started run 821 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:33] Error in run 814: Run already exists in server. Run id(s): {9019123}
[2018-04-11 21:00:33] Started run 822 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:33] Error in run 815: Run already exists in server. Run id(s): {9019306}
[2018-04-11 21:00:33] Error in run 816: Run already exists in server. Run id(s): {9019168}
[2018-04-11 21:00:33] Started run 823 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:33] Started run 824 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:35] Error in run 818: Run already exists in server. Run id(s): {9019162}
[2018-04-11 21:00:35] Error in run 817: Run already exists in server. Run id(s): {9019241}
[2018-04-11 21:00:35] Started run 825 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:35] Started run 826 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:36] Error in run 819: Run already exists in server. Run id(s): {9019174}
[2018-04-11 21:00:36] Started run 827 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:36] Error in run 820: Run already exists in server. Run id(s): {9019179}
[2018-04-11 21:00:36] Error in run 821: Run already exists in server. Run id(s): {9019188}
[2018-04-11 21:00:36] Started run 828 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:36] Started run 829 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:37] Error in run 823: Run already exists in server. Run id(s): {9019236}
[2018-04-11 21:00:37] Error in run 822: Run already exists in server. Run id(s): {9019220}
[2018-04-11 21:00:37] Started run 830 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:37] Error in run 824: Run already exists in server. Run id(s): {9019244}
[2018-04-11 21:00:37] Started run 831 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:37] Started run 832 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:40] Error in run 825: Run already exists in server. Run id(s): {9019254}
[2018-04-11 21:00:40] Error in run 826: Run already exists in server. Run id(s): {9019249}
[2018-04-11 21:00:40] Started run 833 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:40] Started run 834 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:41] Error in run 827: Run already exists in server. Run id(s): {9019257}
[2018-04-11 21:00:41] Started run 835 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:41] Error in run 828: Run already exists in server. Run id(s): {9019293}
[2018-04-11 21:00:41] Error in run 829: Run already exists in server. Run id(s): {9019296}
[2018-04-11 21:00:41] Started run 836 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:41] Started run 837 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:42] Error in run 830: Run already exists in server. Run id(s): {9019311}
[2018-04-11 21:00:42] Error in run 831: Run already exists in server. Run id(s): {9019421}
[2018-04-11 21:00:42] Started run 838 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:42] Error in run 832: Run already exists in server. Run id(s): {9019321}
[2018-04-11 21:00:42] Started run 839 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:42] Started run 840 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:44] Error in run 833: Run already exists in server. Run id(s): {9019323}
[2018-04-11 21:00:44] Error in run 834: Run already exists in server. Run id(s): {9019331}
[2018-04-11 21:00:44] Started run 841 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:44] Started run 842 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:45] Error in run 835: Run already exists in server. Run id(s): {9019361}
[2018-04-11 21:00:45] Started run 843 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:46] Error in run 836: Run already exists in server. Run id(s): {9019418}
[2018-04-11 21:00:46] Error in run 837: Run already exists in server. Run id(s): {9019371}
[2018-04-11 21:00:46] Started run 844 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:46] Started run 845 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:47] Error in run 839: Run already exists in server. Run id(s): {9019383}
[2018-04-11 21:00:47] Error in run 840: Run already exists in server. Run id(s): {9019393}
[2018-04-11 21:00:47] Error in run 838: Run already exists in server. Run id(s): {9019375}
[2018-04-11 21:00:47] Started run 846 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:47] Started run 847 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:47] Started run 848 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:49] Error in run 841: Run already exists in server. Run id(s): {9019407}
[2018-04-11 21:00:49] Started run 849 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:49] Error in run 842: Run already exists in server. Run id(s): {9019427}
[2018-04-11 21:00:49] Started run 850 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:50] Error in run 843: Run already exists in server. Run id(s): {9019475}
[2018-04-11 21:00:50] Started run 851 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:51] Error in run 844: Run already exists in server. Run id(s): {9019442}
[2018-04-11 21:00:51] Error in run 845: Run already exists in server. Run id(s): {9019623}
[2018-04-11 21:00:51] Started run 852 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:51] Started run 853 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:51] Error in run 846: Run already exists in server. Run id(s): {9019459}
[2018-04-11 21:00:52] Error in run 847: Run already exists in server. Run id(s): {9019471}
[2018-04-11 21:00:51] Started run 854 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:52] Error in run 848: Run already exists in server. Run id(s): {9019478}
[2018-04-11 21:00:52] Started run 855 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:52] Started run 856 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:54] Error in run 850: Run already exists in server. Run id(s): {9019486}
[2018-04-11 21:00:54] Error in run 849: Run already exists in server. Run id(s): {9019482}
[2018-04-11 21:00:54] Started run 857 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:54] Started run 858 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:55] Error in run 851: Run already exists in server. Run id(s): {9019514}
[2018-04-11 21:00:55] Started run 859 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:55] Error in run 852: Run already exists in server. Run id(s): {9019524}
[2018-04-11 21:00:55] Error in run 853: Run already exists in server. Run id(s): {9019532}
[2018-04-11 21:00:55] Started run 860 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:55] Started run 861 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:56] Error in run 854: Run already exists in server. Run id(s): {9019537}
[2018-04-11 21:00:56] Started run 862 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:56] Error in run 855: Run already exists in server. Run id(s): {9019547}
[2018-04-11 21:00:56] Error in run 856: Run already exists in server. Run id(s): {9019551}
[2018-04-11 21:00:56] Started run 863 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:56] Started run 864 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:59] Error in run 858: Run already exists in server. Run id(s): {9019578}
[2018-04-11 21:00:59] Error in run 857: Run already exists in server. Run id(s): {9019552}
[2018-04-11 21:00:59] Started run 865 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:00:59] Started run 866 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:00] Error in run 859: Run already exists in server. Run id(s): {9019595}
[2018-04-11 21:01:00] Started run 867 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:00] Error in run 861: Run already exists in server. Run id(s): {9019604}
[2018-04-11 21:01:00] Error in run 860: Run already exists in server. Run id(s): {9019611}
[2018-04-11 21:01:00] Started run 868 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:00] Started run 869 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:01] Error in run 862: Run already exists in server. Run id(s): {9019607}
[2018-04-11 21:01:01] Started run 870 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:01] Error in run 863: Run already exists in server. Run id(s): {9019616}
[2018-04-11 21:01:01] Started run 871 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:01] Error in run 864: Run already exists in server. Run id(s): {9019618}
[2018-04-11 21:01:01] Started run 872 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:03] Error in run 865: Run already exists in server. Run id(s): {9019648}
[2018-04-11 21:01:03] Error in run 866: Run already exists in server. Run id(s): {9019660}
[2018-04-11 21:01:03] Started run 873 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:03] Started run 874 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:04] Error in run 867: Run already exists in server. Run id(s): {9019666}
[2018-04-11 21:01:04] Started run 875 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:05] Error in run 869: Run already exists in server. Run id(s): {9019679}
[2018-04-11 21:01:05] Error in run 868: Run already exists in server. Run id(s): {9019669}
[2018-04-11 21:01:05] Started run 876 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:05] Started run 877 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:05] Error in run 870: Run already exists in server. Run id(s): {9019675}
[2018-04-11 21:01:05] Started run 878 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:06] Error in run 871: Run already exists in server. Run id(s): {9019688}
[2018-04-11 21:01:06] Error in run 872: Run already exists in server. Run id(s): {9019694}
[2018-04-11 21:01:06] Started run 879 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:06] Started run 880 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:08] Error in run 873: Run already exists in server. Run id(s): {9019709}
[2018-04-11 21:01:08] Error in run 874: Run already exists in server. Run id(s): {9019714}
[2018-04-11 21:01:08] Started run 881 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:08] Started run 882 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:09] Error in run 875: Run already exists in server. Run id(s): {9019790}
[2018-04-11 21:01:09] Started run 883 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:10] Error in run 877: Run already exists in server. Run id(s): {9019737}
[2018-04-11 21:01:10] Error in run 876: Run already exists in server. Run id(s): {9019749}
[2018-04-11 21:01:10] Started run 884 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:10] Started run 885 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:10] Error in run 878: Run already exists in server. Run id(s): {9019743}
[2018-04-11 21:01:10] Started run 886 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:11] Error in run 879: Run already exists in server. Run id(s): {9019742}
[2018-04-11 21:01:11] Error in run 880: Run already exists in server. Run id(s): {9019747}
[2018-04-11 21:01:11] Started run 887 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:11] Started run 888 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:13] Error in run 882: Run already exists in server. Run id(s): {9019772}
[2018-04-11 21:01:13] Error in run 881: Run already exists in server. Run id(s): {9019759}
[2018-04-11 21:01:13] Started run 889 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:13] Started run 890 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:14] Error in run 883: Run already exists in server. Run id(s): {9019792}
[2018-04-11 21:01:14] Started run 891 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:14] Error in run 884: Run already exists in server. Run id(s): {9019797}
[2018-04-11 21:01:14] Error in run 885: Run already exists in server. Run id(s): {9019800}
[2018-04-11 21:01:14] Started run 892 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:15] Error in run 886: Run already exists in server. Run id(s): {9019831}
[2018-04-11 21:01:15] Started run 894 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:14] Started run 893 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:15] Error in run 888: Run already exists in server. Run id(s): {9019815}
[2018-04-11 21:01:15] Error in run 887: Run already exists in server. Run id(s): {9019804}
[2018-04-11 21:01:15] Started run 895 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:15] Started run 896 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:18] Error in run 889: Run already exists in server. Run id(s): {9019835}
[2018-04-11 21:01:18] Error in run 890: Run already exists in server. Run id(s): {9019843}
[2018-04-11 21:01:18] Started run 897 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:18] Started run 898 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:19] Error in run 891: Run already exists in server. Run id(s): {9019846}
[2018-04-11 21:01:19] Started run 899 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:19] Error in run 892: Run already exists in server. Run id(s): {9019849}
[2018-04-11 21:01:19] Error in run 893: Run already exists in server. Run id(s): {9019856}
[2018-04-11 21:01:19] Error in run 894: Run already exists in server. Run id(s): {9019911}
[2018-04-11 21:01:19] Started run 900 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:19] Started run 901 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:19] Started run 902 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:20] Error in run 895: Run already exists in server. Run id(s): {9019869}
[2018-04-11 21:01:20] Error in run 896: Run already exists in server. Run id(s): {9019877}
[2018-04-11 21:01:20] Started run 903 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:20] Started run 904 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:22] Error in run 898: Run already exists in server. Run id(s): {9019888}
[2018-04-11 21:01:22] Error in run 897: Run already exists in server. Run id(s): {9019879}
[2018-04-11 21:01:22] Started run 905 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:22] Started run 906 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:23] Error in run 899: Run already exists in server. Run id(s): {9019893}
[2018-04-11 21:01:23] Started run 907 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:24] Error in run 900: Run already exists in server. Run id(s): {9019889}
[2018-04-11 21:01:24] Error in run 901: Run already exists in server. Run id(s): {9019902}
[2018-04-11 21:01:24] Started run 908 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:24] Error in run 902: Run already exists in server. Run id(s): {9019915}
[2018-04-11 21:01:24] Started run 909 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:24] Started run 910 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:25] Error in run 903: Run already exists in server. Run id(s): {9019921}
[2018-04-11 21:01:25] Error in run 904: Run already exists in server. Run id(s): {9019924}
[2018-04-11 21:01:25] Started run 911 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:25] Started run 912 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:27] Error in run 906: Run already exists in server. Run id(s): {9019929}
[2018-04-11 21:01:27] Error in run 905: Run already exists in server. Run id(s): {9019927}
[2018-04-11 21:01:27] Started run 913 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:27] Started run 914 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:28] Error in run 907: Run already exists in server. Run id(s): {9019939}
[2018-04-11 21:01:28] Started run 915 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:29] Error in run 909: Run already exists in server. Run id(s): {9019954}
[2018-04-11 21:01:29] Error in run 908: Run already exists in server. Run id(s): {9019949}
[2018-04-11 21:01:29] Error in run 910: Run already exists in server. Run id(s): {9019959}
[2018-04-11 21:01:29] Started run 916 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:29] Started run 917 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:29] Started run 918 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:30] Error in run 911: Run already exists in server. Run id(s): {9019960}
[2018-04-11 21:01:30] Error in run 912: Run already exists in server. Run id(s): {9019964}
[2018-04-11 21:01:30] Started run 919 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:30] Started run 920 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:32] Error in run 913: Run already exists in server. Run id(s): {9019967}
[2018-04-11 21:01:32] Error in run 914: Run already exists in server. Run id(s): {9019976}
[2018-04-11 21:01:32] Started run 921 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:32] Started run 922 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:33] Error in run 915: Run already exists in server. Run id(s): {9019995}
[2018-04-11 21:01:33] Started run 923 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:33] Error in run 917: Run already exists in server. Run id(s): {9020003}
[2018-04-11 21:01:33] Error in run 916: Run already exists in server. Run id(s): {9019988}
[2018-04-11 21:01:33] Started run 924 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:34] Error in run 918: Run already exists in server. Run id(s): {9020020}
[2018-04-11 21:01:33] Started run 925 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:34] Started run 926 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:34] Error in run 919: Run already exists in server. Run id(s): {9020016}
[2018-04-11 21:01:34] Error in run 920: Run already exists in server. Run id(s): {9020013}
[2018-04-11 21:01:34] Started run 927 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:34] Started run 928 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:37] Error in run 921: Run already exists in server. Run id(s): {9020031}
[2018-04-11 21:01:37] Error in run 922: Run already exists in server. Run id(s): {9020042}
[2018-04-11 21:01:37] Started run 929 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:37] Started run 930 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:38] Error in run 923: Run already exists in server. Run id(s): {9020062}
[2018-04-11 21:01:38] Started run 931 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:38] Error in run 924: Run already exists in server. Run id(s): {9020054}
[2018-04-11 21:01:38] Error in run 926: Run already exists in server. Run id(s): {9020086}
[2018-04-11 21:01:38] Error in run 925: Run already exists in server. Run id(s): {9020074}
[2018-04-11 21:01:38] Started run 932 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:38] Started run 933 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:38] Started run 934 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:39] Error in run 927: Run already exists in server. Run id(s): {9020090}
[2018-04-11 21:01:39] Error in run 928: Run already exists in server. Run id(s): {9020083}
[2018-04-11 21:01:39] Started run 935 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:39] Started run 936 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:41] Error in run 930: Run already exists in server. Run id(s): {9020098}
[2018-04-11 21:01:41] Error in run 929: Run already exists in server. Run id(s): {9020093}
[2018-04-11 21:01:41] Started run 937 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:41] Started run 938 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:42] Error in run 931: Run already exists in server. Run id(s): {9020121}
[2018-04-11 21:01:42] Started run 939 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:43] Error in run 932: Run already exists in server. Run id(s): {9020138}
[2018-04-11 21:01:43] Error in run 933: Run already exists in server. Run id(s): {9020164}
[2018-04-11 21:01:43] Started run 940 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:43] Started run 941 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:43] Error in run 934: Run already exists in server. Run id(s): {9020153}
[2018-04-11 21:01:43] Started run 942 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:44] Error in run 936: Run already exists in server. Run id(s): {9020161}
[2018-04-11 21:01:44] Error in run 935: Run already exists in server. Run id(s): {9020150}
[2018-04-11 21:01:44] Started run 943 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:44] Started run 944 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:46] Error in run 937: Run already exists in server. Run id(s): {9020170}
[2018-04-11 21:01:46] Error in run 938: Run already exists in server. Run id(s): {9020156}
[2018-04-11 21:01:46] Started run 945 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:46] Started run 946 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:47] Error in run 939: Run already exists in server. Run id(s): {9020198}
[2018-04-11 21:01:47] Started run 947 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:48] Error in run 941: Run already exists in server. Run id(s): {9020387}
[2018-04-11 21:01:48] Error in run 940: Run already exists in server. Run id(s): {9020208}
[2018-04-11 21:01:48] Started run 948 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:48] Error in run 942: Run already exists in server. Run id(s): {9020221}
[2018-04-11 21:01:48] Started run 949 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:48] Started run 950 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:49] Error in run 943: Run already exists in server. Run id(s): {9020230}
[2018-04-11 21:01:49] Started run 951 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:49] Error in run 944: Run already exists in server. Run id(s): {9020273}
[2018-04-11 21:01:49] Started run 952 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:51] Error in run 946: Run already exists in server. Run id(s): {9020248}
[2018-04-11 21:01:51] Error in run 945: Run already exists in server. Run id(s): {9020257}
[2018-04-11 21:01:51] Started run 953 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:51] Started run 954 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:52] Error in run 947: Run already exists in server. Run id(s): {9020261}
[2018-04-11 21:01:52] Started run 955 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:52] Error in run 948: Run already exists in server. Run id(s): {9020269}
[2018-04-11 21:01:52] Error in run 949: Run already exists in server. Run id(s): {9020302}
[2018-04-11 21:01:52] Started run 956 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:52] Started run 957 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:53] Error in run 950: Run already exists in server. Run id(s): {9020309}
[2018-04-11 21:01:53] Started run 958 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:53] Error in run 952: Run already exists in server. Run id(s): {9020497}
[2018-04-11 21:01:53] Error in run 951: Run already exists in server. Run id(s): {9020318}
[2018-04-11 21:01:53] Started run 959 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:53] Started run 960 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:56] Error in run 953: Run already exists in server. Run id(s): {9020559}
[2018-04-11 21:01:56] Started run 961 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:56] Error in run 954: Run already exists in server. Run id(s): {9020364}
[2018-04-11 21:01:56] Started run 962 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:57] Error in run 955: Run already exists in server. Run id(s): {9020351}
[2018-04-11 21:01:57] Started run 963 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:57] Error in run 956: Run already exists in server. Run id(s): {9020373}
[2018-04-11 21:01:57] Error in run 958: Run already exists in server. Run id(s): {9020401}
[2018-04-11 21:01:57] Error in run 957: Run already exists in server. Run id(s): {9020396}
[2018-04-11 21:01:57] Started run 964 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:57] Started run 965 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:57] Started run 966 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:58] Error in run 959: Run already exists in server. Run id(s): {9020425}
[2018-04-11 21:01:58] Error in run 960: Run already exists in server. Run id(s): {9020430}
[2018-04-11 21:01:58] Started run 967 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:01:58] Started run 968 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:00] Error in run 961: Run already exists in server. Run id(s): {9020439}
[2018-04-11 21:02:00] Error in run 962: Run already exists in server. Run id(s): {9020460}
[2018-04-11 21:02:00] Started run 969 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:00] Started run 970 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:01] Error in run 963: Run already exists in server. Run id(s): {9020475}
[2018-04-11 21:02:01] Started run 971 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:02] Error in run 964: Run already exists in server. Run id(s): {9020479}
[2018-04-11 21:02:02] Error in run 965: Run already exists in server. Run id(s): {9020491}
[2018-04-11 21:02:02] Started run 972 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:02] Started run 973 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:02] Error in run 966: Run already exists in server. Run id(s): {9020499}
[2018-04-11 21:02:02] Started run 974 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:03] Error in run 968: Run already exists in server. Run id(s): {9020534}
[2018-04-11 21:02:03] Error in run 967: Run already exists in server. Run id(s): {9020508}
[2018-04-11 21:02:03] Started run 975 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:03] Started run 976 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:05] Error in run 969: Run already exists in server. Run id(s): {9020546}
[2018-04-11 21:02:05] Error in run 970: Run already exists in server. Run id(s): {9020550}
[2018-04-11 21:02:05] Started run 977 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:05] Started run 978 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:06] Error in run 971: Run already exists in server. Run id(s): {9020556}
[2018-04-11 21:02:06] Started run 979 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:07] Error in run 973: Run already exists in server. Run id(s): {9020677}
[2018-04-11 21:02:07] Error in run 974: Run already exists in server. Run id(s): {9020576}
[2018-04-11 21:02:07] Error in run 972: Run already exists in server. Run id(s): {9020648}
[2018-04-11 21:02:07] Started run 980 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:07] Started run 981 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:07] Started run 982 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:08] Error in run 975: Run already exists in server. Run id(s): {9020608}
[2018-04-11 21:02:08] Error in run 976: Run already exists in server. Run id(s): {9020619}
[2018-04-11 21:02:08] Started run 983 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:08] Started run 984 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:10] Error in run 978: Run already exists in server. Run id(s): {9020628}
[2018-04-11 21:02:10] Error in run 977: Run already exists in server. Run id(s): {9020613}
[2018-04-11 21:02:10] Started run 985 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:10] Started run 986 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:11] Error in run 979: Run already exists in server. Run id(s): {9020622}
[2018-04-11 21:02:11] Started run 987 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:11] Error in run 980: Run already exists in server. Run id(s): {9020654}
[2018-04-11 21:02:11] Error in run 981: Run already exists in server. Run id(s): {9020667}
[2018-04-11 21:02:11] Started run 988 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:12] Error in run 982: Run already exists in server. Run id(s): {9020685}
[2018-04-11 21:02:11] Started run 989 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:12] Started run 990 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:12] Error in run 984: Run already exists in server. Run id(s): {9020795}
[2018-04-11 21:02:12] Error in run 983: Run already exists in server. Run id(s): {9020679}
[2018-04-11 21:02:12] Started run 991 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:12] Started run 992 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:15] Error in run 985: Run already exists in server. Run id(s): {9020702}
[2018-04-11 21:02:15] Error in run 986: Run already exists in server. Run id(s): {9020835}
[2018-04-11 21:02:15] Started run 993 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:15] Started run 994 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:16] Error in run 987: Run already exists in server. Run id(s): {9020728}
[2018-04-11 21:02:16] Started run 995 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:16] Error in run 988: Run already exists in server. Run id(s): {9020733}
[2018-04-11 21:02:16] Error in run 990: Run already exists in server. Run id(s): {9020744}
[2018-04-11 21:02:16] Error in run 989: Run already exists in server. Run id(s): {9020739}
[2018-04-11 21:02:16] Started run 996 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:16] Started run 998 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:16] Started run 997 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:17] Error in run 991: Run already exists in server. Run id(s): {9020851}
[2018-04-11 21:02:17] Error in run 992: Run already exists in server. Run id(s): {9020769}
[2018-04-11 21:02:17] Started run 999 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:17] Started run 1000 on task 9981, dataset 'cnae-9'.
[2018-04-11 21:02:20] Error in run 993: Run already exists in server. Run id(s): {9020791}
[2018-04-11 21:02:20] Error in run 994: Run already exists in server. Run id(s): {9020801}
[2018-04-11 21:02:20] Error in run 995: Run already exists in server. Run id(s): {9020814}
[2018-04-11 21:02:21] Error in run 996: Run already exists in server. Run id(s): {9020821}
[2018-04-11 21:02:21] Error in run 998: Run already exists in server. Run id(s): {9020865}
[2018-04-11 21:02:21] Error in run 997: Run already exists in server. Run id(s): {9020840}
[2018-04-11 21:02:22] Error in run 1000: Run already exists in server. Run id(s): {9020872}
[2018-04-11 21:02:22] Error in run 999: Run already exists in server. Run id(s): {9020868}
